<html><head><title>Interactive Fly, Drosophila</title>



<meta name="keywords" content="Interactive fly, Drosophila, developmental biology, hunchback, zinc finger, gap gene, anterior/posterior axis, central nervous system, cns, neuroblast, repressor"></head><body bgcolor="#ffffff">

 <i>hunchback</i>:<font size="-1">

<a href="http://www.sdbonline.org/fly/segment/hunchbk1.htm#Bio">Biological Overview</a> 

| <a href="http://www.sdbonline.org/fly/segment/hunchbk2.htm#EvHoms">Evolutionary Homologs</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk3.htm#Regul">Regulation</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk3.htm#TofA">Targets of activity</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk4.htm#PROT">Protein Interactions</a> 

| <a href="http://www.sdbonline.org/fly/segment/hunchbk8.htm#POST">Post-transcriptional Regulation</a> 







| <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#DevBio">Developmental Biology</a> 

 | <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#EfMu">Effects of Mutation</a>



 | <a href="http://www.sdbonline.org/fly/segment/hunchbk6.htm#Ref">References</a></font>

<hr>

<table width="560" cellpadding="0" cellspacing="0" border="0">

<tbody><tr>

<td width="300" valign="top">

<p><strong>   Gene name - <i>hunchback</i></strong>           

</p><p><strong>   Synonyms  </strong> -  

</p><p><strong>   Cytological map position - </strong>85A3-B1 



</p><p><strong>   Function - </strong>transcription factor



</p><p><strong>   Keywords - </strong>morphogen - anterior-posterior axis

and <a href="http://www.sdbonline.org/fly/aignfam/gapnprl.htm">gap gene</a>

</p>



</td><td width="250" valign="top">







<p><strong> Symbol - <i> hb         </i> </strong>



</p><p><strong> FlyBase ID:</strong><a href="http://flybase.bio.indiana.edu/.bin/fbidq.html?FBgn0001180"> FBgn0001180 </a>





</p><p><strong>  Genetic map position - </strong> 3-48.3

            

</p><p><strong>  Classification -  </strong><a href="http://www.sdbonline.org/fly/aimain/6zinc.htm">zinc finger protein</a> (C2H2)



</p><p><strong>  Cellular location - </strong>nuclear





</p></td> </tr>



</tbody></table>





<br> <hr>   <b>NCBI and GadFly links:</b> 





   <a href="http://www.ncbi.nlm.nih.gov/sutils/blink.cgi?pid=15291987&amp;all=1">Precomputed BLAST</a>

    |  <a href="http://www.ncbi.nlm.nih.gov/LocusLink/LocRpt.cgi?l=41032">LocusLink</a>

    |   <a href="http://flybase.bio.indiana.edu/.bin/fbidq.html?FBan0009786" class="RR" alt="GadFly">GadFly</a>



<br><br>   <b>Fly GRID</b>: <a href="http://biodata.mshri.on.ca/fly_grid/servlet/SearchResults?keywords=FBgn0001180">Hunchback</a> genetic and physical interactions 

<hr>

<dl><dt><a name="Bio"><strong></strong></a><center><a name="Bio"><strong>BIOLOGICAL OVERVIEW</strong></a></center><a name="Bio"> </a></dt><dd><!--The Interactive Fly &#169; 1997 Thomas B. Brody--><p>





</p><p><a name="Bio">The function of <i> hunchback</i> is central to the
establishment of an anterior-posterior gradient of gene activity in the
transition from   unfertilized egg to  developing zygote.  As its name
suggests, <i>hunchback</i> has a special role in the development of the trunk (thorax) of the fly.



</a></p><p><a name="Bio">Maternal <i>HB</i> mRNA, is intitially
distributed evenly throughout the egg.    Nanos, whose mRNA is
localized to the posterior pole of mature oocyte, functions to inhibit <i>Hunchback</i>: the Nanos protein inactivates <i>HB</i>
mRNA, preventing its translation in the posterior. Thus Nanos, through
its inhibition of HB translation, establishes a concentration gradient
of maternally derived HB protein complementary to the gradient of </a><a href="http://www.sdbonline.org/fly/torstoll/nanos1.htm">Nanos</a> protein (Pelegri, 1994).  It is not Nanos itself that binds to the Nanos response elements of <i>HB</i> mRNA, but  rather another protein, Pumilio,  that apparently recruits Nanos into a multiprotein-RNA complex (Murata, 1995).



</p><p>After fertilization,   maternally derived Hunchback is
supplanted by a zygotic HB transcript. Transcription is driven by
Bicoid in the anterior. <a href="http://www.sdbonline.org/fly/segment/bicoid1.htm">Bicoid</a> is arrayed in an anterior to posterior gradient, and activates <i>hunchback</i> expression along this gradient, giving rise to an anterior-posterior <i>Hunchback </i>zygotic gradient.



 </p><p> Hunchback acts both to activate anterior <a href="http://www.sdbonline.org/fly/aignfam/gapnprl.htm">gap</a>
gene function as a co-activator with Bicoid, and to shift the effective
morphogenetic activity of Bicoid toward the posterior, thus extending
the effective range of Bicoid (Simpson-Brose, 1994). Hunchback can
operate both as a transcription activator or repressor, and as such
determines the placement of both anterior and posterior gap genes. 
Hunchback's main role is as a repressor of posterior gap gene
expression in the anterior. <i><a href="http://www.sdbonline.org/fly/segment/kruppel1.htm">Krüppel</a></i> expression in the middle of the embryo is regulated by HB. <i> <a href="http://www.sdbonline.org/fly/segment/knirps1.htm">knirps</a></i> and<i> <a href="http://www.sdbonline.org/fly/segment/giant.htm">giant</a></i> are expressed in the posterior, but these genes are repressed in the anterior by Hunchback. 



 </p><p><a href="http://www.sdbonline.org/fly/polycomb/enhzeste.htm">Enhancer of zeste</a>( E[z]) is required to maintain transcriptional repression of <i>knirps</i> and<i> giant </i>once repression has been initiated by Hunchback.  A role for <a href="http://www.sdbonline.org/fly/aignfam/polycomb.htm">Polycomb group</a>
genes in the regulation of gap genes is a fairly recent idea; it is now
apparent that Hunchback and E(z) act together at the same cis-acting
sequences to establish repression in the <i>knirps </i>promoter (Pelegri, 1994).



</p><p>Hunchback activity in the posterior is regulated by <a href="http://www.sdbonline.org/fly/torstoll/tailless.htm">Tailless</a> and <a href="http://www.sdbonline.org/fly/gene/huckebin.htm">Huckebein</a> (Margolis, 1995). <i> hunchback</i> acts like a gap gene in the posterior.  Mutants evince fused 7th and 8th segments <a href="http://www.sdbonline.org/fly/aimain/images.htm">[Images]</a>
(Tautz, 1987).  Perhaps Hunchback acts as a cofactor with Krüppel and
Knirps.  It has been demonstrated that HB can associate with these gap
gene products and that their interaction results in gene repression
(Sauer, 1995). </p><p>While the Bcd gradient has served as a model system in
understanding pattern formation in Drosophila, it is suspected that
this is not the case in more ancestral insects. The long-germ mode of
development as found in Drosophila is probably an adaptation to its
particularly rapid embryogenesis. The ancestral type of embryogenesis
in insects and arthropods is the short germ type. In these embryos, the
germ rudiment forms at the posterior ventral side of the egg. In
extreme cases like the grasshopper, it may be restricted to only a few
percent of the total egg length - which makes it difficult to imagine
how an anteriorly localized <i>BCD</i> mRNA could determine pattern
formation at the posterior end of the egg. Moreover, classical
experiments have only yielded evidence for a posteriorly localized
organizing activity. Therefore, <i>bcd</i> could be considered a late
addition during insect evolution and its pivotal function during
embryogenesis could be restricted to higher dipterans. This paper is
concerned with early pattern formation of the flour beetle <i>Tribolium castaneum</i>.
Tribolium is a typical example for short germ embryogenesis,
representing the ancestral type of embryogenesis in insects, albeit not
in its extreme form, like the grasshopper (see <a href="http://www.sdbonline.org/fly/vdevlhom/trib.htm">Tribolium early embryonic development</a>).
In contrast to Drosophila, only cephalic and thoracic segments, but not
abdominal segments, are determined during the blastoderm stage.
Furthermore, the most anterior 20% of the Tribolium blastoderm cells
form an extra-embryonic membrane, the serosa. This structure is not
found in this form in higher Dipterans like Drosophila, but is again an
ancestral feature of insect embryogenesis. Prior to gastrulation, most
blastoderm cells move from anterior and dorsal positions towards the
posterior ventral region where they form the embryo proper. This germ
rudiment then continues to grow from its posterior end to form a germ
band which eventually encompasses all abdominal segments (Wolff, 1998).</p><p>Thus, in short germ embryos, the germ rudiment forms at the
posterior ventral side of the egg, while the anterior-dorsal region
becomes the extra-embryonic serosa. It is difficult to see how in these
embryos an anterior gradient like that of Bicoid protein in <i>Drosophila</i> could  be directly involved in patterning of the germ rudiment. Moreover, since it has not yet been possible to recover a <i>bicoid</i>
homolog from any species outside the diptera, it has been speculated
that the anterior Bicoid gradient could be a late addition during
insect evolution. This question was addressed by analyzing the
regulation of potential target genes of <i>bicoid</i> in the short germ embryo of <i>Tribolium castaneum</i>.  Homologs of <i>caudal</i> and <i>hunchback</i> from Tribolium are regulated by Drosophila <i>bicoid</i>. In Drosophila, maternal <i>Caudal</i> mRNA is translationally repressed by Bicoid. Tribolium <i>Caudal</i>
RNA is also translationally repressed by Bicoid, when it is transferred
into Drosophila embryos under a maternal promoter. This strongly
suggests that a functional <i>bicoid</i> homolog must exist in Tribolium. The second target gene, <i>hunchback</i>, is transcriptionally activated by Bicoid in <i>Drosophila</i>. Transfer of the regulatory region of <i>Tribolium hunchback</i> into <i>Drosophila</i> also results in regulation by early maternal factors, including Bicoid, but in a pattern that is more reminiscent of Tribolium <i>hunchback</i>
expression, namely in two early blastoderm domains. Using enhancer
mapping constructs and footprinting, it has been shown that Caudal
activates the posterior of these domains via a specific promoter. These
experiments suggest that a major event in the evolutionary transition
from short to long germ embryogenesis was the switch from activation of
the <i>hunchback</i> gap domain by Caudal to direct activation by
Bicoid. This regulatory switch can explain how this domain shifted from
a posterior location in short germ embryos to its anterior position in
long germ insects, and it also suggests how an anterior gradient can
pattern the germ rudiment in short germ embryos, i.e. by regulating the
expression of <i>caudal</i> (Wolff, 1998).  



</p><p>The key to understanding the qualitative switch that took place
in insect evolution is believed to lie in the more anterior serosa
expression domain of Tribolium <i>hb</i>.  Reporter gene data suggest
that this domain may already be activated by Bcd in Tribolium. To
explain the switch in the regulation of the more posterior gap domain
of <i>hb</i> expression, one can envision an intermediate state, where
the serosa domain and the embryonic (gap) domain have fused into a
single domain. To achieve this, the evolution of a few additional Bcd
binding sites in the <i>hb</i> upstream region would have been
sufficient. In this intermediate stage both Bcd and Cad could have
acted as activators on the gap domain of hb. Subsequent loss of Cad
regulation would then have moved the posterior boundary of this
combined domain towards the anterior. It is noted that the Tribolium <i>hb</i>
gene has three known promoters, one of which appears to be specialized
for mediating  Cad regulation. In Drosophila, only two promoters are
present, neither of which has a known responsiveness to Cad. Thus, in
all likelihood, the Cad dependent promoter and its associated enhancer
was lost.   Since no other enhancer activity has been found for later
expression patterns of <i>hb</i> in the <i>cad</i> dependent fragment,
the loss of this region could have been a single step. Intriguingly, a
combined serosa and gap domain is still evident in the lower dipteran
Clogmia. In this fly, <i>hb</i> is expressed in a large anterior
domain, from which at later stages also the serosa is recruited (Rohr,
personal communication to Wolff, 1998). This mechanism, the
modification of the way  gap genes sense maternal positional
information while this information itself remains constant, can explain
how the blastoderm fate map changed during evolution of short germ
insects to insects with long germ embryos. Moreover, it represents an
intriguing example for the importance of regulatory adaptation during
the evolution of developmental processes (Wolff, 1998). </p><p>In addition to its early regulatory functions during
segmentation, Hunchback is also expressed in the developing nervous
system (see <a href="http://www.sdbonline.org/fly/vdevlhom/image1.htm#cns">Lateral views of Drosophila CNS</a>). One possible CNS regulatory target for Hb is the POU gene <a href="http://www.sdbonline.org/fly/gene/pdm-1.htm"><i>pdm-1</i></a>.  Hb regulates <i>pdm-1</i>
expression at the cellular blastoderm stage (Lloyd, 1991; Cockerill,
1993), and may play a similar role in the CNS.  Since Hb and <a href="http://www.sdbonline.org/fly/neural/ming.htm">Castor</a>
bind similar promoter target sequences, an exploration was carried out
of the embryonic distribution of the three proteins using polyclonal
antibodies.  It is suggested that Hb and Cas act in a cooperative,
non-overlapping manner to control POU gene expression during Drosophila
CNS development.  By silencing <i>pdm</i> expression in early and late
NB sublineages, Hb and Cas establish three pan-CNS compartments whose
cellular constituents are marked by the expression of either Hb, Pdm,
or Cas.   During the initial S1 and S2 waves of NB delaminations, Pdm-1
is expressed in most, if not all, neuroectoderm cells.  However, no
Pdm-1 is detected in fully delaminated NBs and during stage 9 only a
small subset of ventral cord GMCs express detectable levels.  At this
time, Hb expression is detected in all fully delaminated NBs and in
many of their GMCs but not in neuroectoderm cells.  Starting at late
stage 9, Hb immunoreactivity is progressively lost from NBs; by late
stage 10 only a small subset of ventral cord NBs express Hb.  However,
Hb is detected in many GMC and in their progeny generated during the
first rounds of GMC production.  These early sublineages reside
predominantly along the inner/dorsal surfaces of the developing
ganglia.  The reduction in Hb NB expression coincides with the
activation of Pdm-1 NB expression; by late stage 10, Pdm-1 is detected
in many cephalic lobe (see <a href="http://www.sdbonline.org/fly/vdevlhom/image1.htm#cephalic">Views of cephalic lobe neuroblasts</a>)
and ventral cord NBs and in GMCs.  Similar to the dynamics of Hb
expression, Pdm-1 NB expression is transient.  However, many GMCs and
their progeny arising from the Pdm-expressing NBs maintain high levels
of Pdm-1. (Kambadur, 1998). </p><p></p></dd></dl><hr><dl><dt><p><strong><center>GENE STRUCTURE</center></strong>

</p><p></p></dt><dd> There are two transcripts, P1 (maternal &amp;
zygotic) and P2 (zygotic), initiated from different promoters.  The P2
site of initiation is within the  DNA coding for the large intron of
the maternal transcript.  There is a small zygotic intron which shares
a common acceptor site with the maternal intron (Tautz, 1987).<p>

</p></dd></dl>

<strong> cDNA clone length - </strong> 3.2 kb for P1 and 2.9 kb for P2          <p>

<strong>Bases in 5' UTR  -</strong> 221 for P1 and 152 for P2          </p><p>

<strong> Exons - </strong>two      </p><p></p><hr><dl><dt><strong><center>PROTEIN STRUCTURE</center></strong><p></p></dt><dd>



</dd><dt><strong>Amino Acids - </strong> 758                </dt><dd><p>

</p></dd><dt><strong>Structural Domains</strong></dt><dd><p>Hunchback
and Krüppel are homologous; they share four zinc finger domains. 
Hunchback has a higher molecular weight than Krüppel, because of an
additional two zinc fingers at its C-terminal end (Tautz, 1987).   A
subset of zinc finger transcription factors contain amino acid
sequences that resemble those of Krüppel.  They are characterized by
multiple zinc fingers containing the conserved sequence
CX2CX3FX5LX2HX3H (X is any amino acid, and the cysteine and histidine
residues are involved in the coordination of zinc) that are separated
from each other by a highly conserved 7-amino acid inter-finger spacer,
TGEKP(Y/F)X, often referred to as the H/C link. </p><p>Each 30-residue zinc finger motif folds to form an independent
domain with a single zinc ion tetrahedrally coordinated beween an
irregular, antiparallel, two stranded ß-sheet and a short alpha-helix. 
Each zinc finger of mouse Zif268 (which has three fingers) binds to DNA
with the amino terminus of its helix angled down into the major groove.
An important contact between the first of the two histidine zinc
ligands and the phosphate backbone of the DNA contributes to fixing the
orientation of the recognition helix.  Although the two fingers of
Drosophila Tramtrack interact with DNA in a way very similar to those
of Zif268, there are important differences.  Tramtrack has an
additional amino-terminal ß-strand in the first of the three zinc
fingers.  The charge-relay zinc-histidine-phosphate contact of Zif268
is substituted by a tyrosine-phosphate contact.  In addition, for TTK,
the DNA is somewhat distorted with two 20 degree bends.  This
distortion is correlated with changes from the rather simple periodic
pattern of amino base contacts seen in Zif268 and finger 2 of TTK
(Klug, 1995 and references). </p><p><a href="http://www.sdbonline.org/fly/neural/ming.htm">Castor</a>,
a transcription factor  with similar DNA binding specificity to that of
Hb, contains a centrally located Zn-finger domain made-up of four
consecutive C2-H2C2-H2 repeats.  The second C2-H2 of each repeat
closely resembles fingers of the Xenopus TFIIIA C2-H2 class.  Flanking
this repeat are motifs that may constitute either transcription
transactivation or repression domains.   UV induced protein-DNA
cross-linking in vivo studies reveal that Cas binds genomic DNA.  To
determine if Cas is a sequence-specific DNA-binding protein, the cyclic
amplification of selected targets protocol was used.  After six rounds
of selection/amplification, sequencing of cloned fragments revealed 
that all had at least one sequence motif in common and some contained
two core recognition sequences.  DNA fragments containing one site
homologous to the consensus site produce  a single prominent Cas-DNA
gel-shift; a fragment with two, generates two complexes.  Addition of
Cas-specific antisera causes a super-shift of the Cas-DNA complex.  A
search of known transcription factor DNA-binding sites shows that the
Cas recognition sequence is almost identical to that of the Drosophila
Zn-finger protein Hunchback.  The Cas consensus matches 9 out of 10 bp
for the reported Hb sites.  To determine if Cas binds Hb sites,
gel-shift experiments using DNA fragments were carried out with exact
sequence matches to Hb targets.  Cas does indeed bind to these sites. 
The sequence-specificity of Cas-DNA binding to Hb recognition sites was
further tested by competition assays and base-pair substitutions. 
Taken together, these experiments demonstrate that Cas can bind to the
same DNA sites as Hb, raising the possibility that it modulates
transcriptional activities of genes also regulated by Hb. Secondary
structure predictions of the Cas finger domain indicate that only the
first and third of its TFIIIA-like fingers contain alpha-helices. 
Interestingly, optimal alignment of Cas and Hb fingers reveals that the
first and third a-helices of Cas share the highest homology with the
corresponding a-helices of Hb (33% identity for the first and 27% for
the third).  Although speculative, their shared DNA-binding preferences
may  be due in part to the shared residues found in these predicted
reading heads.  Outside of their Zn-fingers, Hb and Cas show no obvious
sequence similarities (Kambadur, 1998). </p><p></p></dd></dl><hr>

<!--Copy links here--> 

 <i>hunchback</i>:<font size="-1">

<a href="http://www.sdbonline.org/fly/segment/hunchbk1.htm#Bio">Biological Overview</a> 

| <a href="http://www.sdbonline.org/fly/segment/hunchbk2.htm#EvHoms">Evolutionary Homologs</a> 





| <a href="http://www.sdbonline.org/fly/segment/hunchbk3.htm#Regul">Regulation</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk7.htm#TofA">Targets of activity</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk4.htm#PROT">Protein Interactions</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk8.htm#POST">Post-transcriptional Regulation</a> 





| <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#DevBio">Developmental Biology</a> 

 | <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#EfMu">Effects of Mutation</a>



 | <a href="http://www.sdbonline.org/fly/segment/hunchbk6.htm#Ref">References</a>



<pre>date revised: 1 Oct 98 </pre></font> 



<p><a href="http://www.sdbonline.org/fly/aimain/1aahome.htm">Home page</a>: The Inter<i>active</i> Fly © 1995, 1996 Thomas B. Brody, Ph.D.</p><p> Please e-mail comments/corrections to <a href="mailto:brodyt@codon.nih.gov"><em>brodyt@codon.nih.gov</em></a>

<br>

</p><p><a href="http://www.sdbonline.org/"><img src="hunchbk1_files/SDBSmLogo.gif" alt="SDB Logo" hspace="3" vspace="3" height="50" width="50" align="middle">SDB home page</a> || <a href="http://www.sdbonline.org/google.search.html">Search the SDB site </a> 

</p></body></html>