<html><head><title>Interactive Fly, Drosophila</title></head>

<body bgcolor="#ffffff"><font size=""></font>



<p>

<a name="Regul"><i>hunchback</i> 

</a></p><hr>







<p><a name="Regul"><strong><center> REGULATION</center></strong></a></p><p></p><dl><dt><a name="Promot"> <strong>  Promoter Structure  </strong></a></dt><dd>



<p><a name="Promot">There are two promoter regions.  The posterior
regulatory region is several kb upstream of the P1 (maternal &amp;
zygotic) transcript.  The anterior regulatory region is upstream of the
P2 (zygotic) transcript within the large maternal intron. </a></p><p><a name="Promot"><i>hunchback</i> expression in the tail
initiates from two promoters.  A cap, covering the terminal 15% of the
embryo, is composed only of mRNA from the distal transcription
initiation site (P1).  A  posterior stripe generated later is composed
of mRNA from both the distal and proximal (P2) transcription
initiation sites,  upstream of the P2 site of initiation. The
posterior regulation region, a 1.4 kb fragment of the <i>hb</i> upstream region is both necessary and sufficient for posterior expression. Sequences within this fragment

mediate activation by the terminal gap gene<i> </i></a><i><a href="http://www.sdbonline.org/fly/torstoll/tailless.htm">tailless</a></i> and repression by terminal gap gene <i><a href="http://www.sdbonline.org/fly/gene/huckebin.htm">huckebein</a></i>, which together direct the

formation of the posterior<i> hb</i> stripe.  The TLL protein binds in vitro to specific sites

within the 1.4 kb posterior enhancer region, providing the first direct evidence for activation of

gene expression by TLL (Margolis, 1995). 







</p><p>In adult Drosophila females maternal transcripts of <i>hunchback</i>

 are produced only by the distal (P1) promoter.  This expression is largely restricted to the <a href="http://www.sdbonline.org/fly/aimorph/oocyte.htm">ovarian nurse cells</a>. A deletion analysis of the <i>hb</i> promoter

using lacZ reporter constructs defines a 1.2-kb genomic DNA fragment surrounding the P1

promoter  sufficient to reproduce the wild-type pattern of <i>hb</i> ovarian transcript accumulation (Margolis, 1994).











</p><p>The maternal determinant Bicoid (Bcd) represents the paradigm of a morphogen that provides

positional information for pattern formation. However, since <i>bicoid</i>
seems to be a recently acquired gene in flies, the question has been
raised as to how embryonic patterning is achieved in organisms with
more ancestral modes of development. Because the phylogenetically
conserved Hunchback (Hb) protein acts as a morphogen in abdominal
patterning, it was asked which functions of Bcd could be performed by
Hb. By reestablishing a proposed ancient regulatory circuitry in which
maternal Hb controls zygotic <i>hunchback</i> expression, it has been shown that Hb is able to form thoracic segments in the absence of Bcd (Wimmer, 2000). 







</p><p>A functional <i>hb</i> transgene has been generated that is

missing all P2 promoter sequences and relies solely on the P1 promoter  (<i>hbP1only</i>).  <i>hbP1only</i>

constructs do not respond to <i>bcd</i> and do not

mediate gene expression in the anterior cap domain. Therefore, <i>hbP1only</i> uncouples the direct link between the

Bcd and Hb morphogen systems. Zygotic <i>hb</i> mutants derived

from heterozygous parents do not develop labial

or thoracic structures, and they also show a fusion of abdominal

segments A7 and A8. When one copy of the <i>hbP1only</i> transgene is provided

zygotically (by the father) to a <i>hb</i> mutant embryo,  it

rescues the posterior phenotype, and A7 and A8 developed normally. The labial/thoracic phenotype is not rescued. However, when

<i>hbP1only</i> is provided as one copy by the mother to a

<i>hb</i> mutant embryo, the posterior and part of the anterior phenotype are rescued. These embryos exhibit normal labial and

prothoracic (T1) segments, and only lack meso- and meta-thoracic segments (T2 and T3). The anterior

rescue is due to the maternal contribution of <i>hbP1only</i>

because sibling embryos that do not inherit the <i>hbP1only</i>

construct zygotically also exhibit the partial anterior (but not the

posterior) rescue. This indicates that restoring high levels

of maternal <i>hb</i> expression (i.e., two copies: one wild type

plus one copy of <i>hbP1only</i>) is sufficient to rescue the

labial and prothoracic segments in the zygotic <i>hb</i> mutant

progeny. Therefore, the lack of zygotic

<i>hb</i>   leads only to the loss of T2 and T3 and to the

fusion of A7 and A8, whereas the previously reported zygotic

<i>hb</i> phenotype represents a

combination of a haploinsufficient maternal plus a zygotic phenotype  (Wimmer, 2000).







</p><p>The loss of zygotic <i>hb</i> activity affects regions of the

embryo that correspond to the two late stripes of zygotic <i>hb</i>
expression: The A7-A8 fusion corresponds to the posterior stripe,
whereas the loss of T2 and T3 corresponds to the PS4 stripe, which
starts as a fairly wide domain covering the anlagen of T2 and T3. This
correlation between the zygotic <i>hb</i> phenotype and the late stripe expression pattern led to a reconsideration of  the importance of the early <i>bcd</i>-dependent anterior cap domain. Under some conditions, <i>hbP1only</i>

(maternal <i>hb</i> contribution plus stripe expression) might

suffice for normal segmentation of head and thorax, making superfluous

the<i> bcd</i>-dependent anterior cap domain. Hence, the <i>hb</i> PS4 stripe is activated without <i>bcd</i>-dependent

<i>hb</i> expression<i>.</i> This stripe is repressed by

the <i>knirps </i>abdominal gap-gene product and is

activated by high levels of Hb itself, either directly or indirectly

(through repression of <i>kni</i>). Embryos that lack the <i>bcd</i>-dependent

<i>hb</i> cap domain have been generated that contain  an increased maternal

<i>hb</i> contribution (to four copies) and <i>kni</i> reduced

to one copy. These embryos display a range of

partially rescued <i>hb</i> phenotypes, including some embryos with a full set of head and thoracic segments. Thus,

<i>bcd</i>-dependent <i>hb</i> expression can in principle be

dispensable for embryonic segmentation, and the only critical anterior

domain of zygotic <i>hb</i> expression appears to be the PS4

stripe, with the <i>bcd</i>-dependent cap domain serving to

activate this stripe. This role is likely achieved by the maternal

<i>hb</i> contribution in species where zygotic <i>hb</i> is not under the control of <i>bcd</i> or where a <i>bcd</i>

homolog might not exist  (Wimmer, 2000). 





</p><p>The rescue of T2 and T3 structures by <i>bcd</i>-independent

<i>hb</i> expression raises the question of whether these

structures could develop in a completely <i>bcd</i>-independent

manner. Embryos derived from <i>bcd</i> mutant mothers develop

ectopic tail structures that replace head and thorax and exhibit a

disruption of some abdominal segments. Although previous work

 has shown that, in the absence of

<i>bcd</i>, high levels of maternal <i>hb</i> can rescue a

normal abdomen and some thoracic structures, no complete thoracic

segments can be induced. A <i>bcd</i>-independent source

for high levels of zygotic <i>hb</i> expression was introduced into a <i>bcd</i> mutant background. By establishing this artificial zygotic Hb

gradient, two notable results were obtained, with variable expressivity:

(1) about 20% of the embryos exhibit rescued T2 and T3 segments. The maintenance of high Hb levels that lead to the rescue of

thoracic segments is likely due to the activation of the <i>hb</i>

stripe element because the <i>hbP1AB </i>reporter
is activated as a stripe where T2 and T3 form.  (2) Most of the ectopic
tail structures that are anteriorly duplicated in <i>bcd</i> mutants
are suppressed, suggesting further redundancy between Hb and Bcd.
However, Hb and Bcd must act at different levels in suppressing these
tail structures, which depend on the activity of the <i>caudal </i>(<i>cad</i>) gene: Bcd acts by repressing <i>cad</i> mRNA translation, whereas Hb does not but might instead

interfere with Cad protein function. This <i>bcd</i>-independent

suppression of <i>cad</i> function might be important in organisms

where the Cad gradient only forms late and represents

another variation as to how <i>cad</i> activity is suppressed at

the anterior of the embryo  (Wimmer, 2000).





</p><p>Different species use various strategies for repression of Cad

function: In Drosophila, translational repression of

<i>CAD</i> mRNA involves the Bcd homeoprotein,

whereas in <i>Caenorhabditis elegans</i> repression involves the

KH-domain protein MEX-3. In vertebrates, a mutually

antagonistic relation between <i>otd</i>-like and

<i>cad</i>-like genes has been proposed to reflect an ancestral

system to pattern the anteroposterior axis of the embryo. In arthropods, ancestral head determinants are

probably encoded by <i>otd-</i>like genes as well. Thus, in the

beetle <i>Tribolium</i>, where no <i>bcd</i> homologs but

Bcd-like activities have been found, these activities

are probably also covered by Otd or KH-domain proteins. This is

consistent with the Otd-like DNA binding specificity of Bcd, which is

atypical for a factor encoded by a gene duplication in the Hox cluster.

This change in specificity was probably crucial for Bcd to acquire its

key role in anterior patterning, because it allowed Bcd to function both as

an RNA binding protein and as an Otd-like transcription

factor. In this respect, it is not surprising that the zinc-finger

protein Hb cannot completely replace Bcd in the head region. Even the

highest levels of Hb obtained in these experiments were not able to

induce head formation in the absence of Bcd. However, Hb is required

for the posterior head region (maxillary and labial segment) and supports anterior head development by synergizing

with Bcd. It will be interesting to see whether such a

synergism can also take place between Hb and other more ancestral head

determinants  (Wimmer, 2000).  





</p><p>These results indicate that the two morphogenetic systems,

Bcd and Hb, do not need to be directly linked. Hence, the direct

regulation of <i>hb</i> by Bcd might represent a recent

evolutionary addition to the insect body plan. In

Drosophila, the abundance of<i> bcd</i>-dependent

<i>hb</i> expression eventually renders superfluous the maternal

<i>hb</i> contribution, which is widespread within

arthropods. Consistent with the idea that the

<i>bcd</i>-dependent <i>hb</i> expression represents a recent evolutionary acquisition, the P2 promoter contains only activator sites

that allow the direct response to a specific threshold level of a

morphogen. This might be a unique situation,

given that most other developmentally regulated promoters contain, in

addition to activator sites, repressor elements for setting the exact

borders of gene expression. By tinkering with the

rather plastic mechanisms of early development, the

ontogeny of Drosophila could be changed toward an inferred ancestral state

where maternal Hb controls zygotic <i>hb</i>. This change could be

brought about by altering patterns and levels of gene expression;  this

presents the most likely variation on which evolutionary processes are

based  (Wimmer, 2000). 











</p><p></p></dd><dt> <a name="TransReg"><strong> Transcriptional Regulation </strong></a></dt><dd><p><a name="TransReg"> 







</a></p><p><a href="http://www.sdbonline.org/fly/segment/bicoid1.htm">Bicoid</a> activates <i>hunchback</i>'s anterior to posterior zygotic gradient (Tautz, 1988 and Struhl, 1989).



</p><p>Bcd contains three putative activation domains: a glutamine-rich
region, which interacts in vitro with TAFII110; an alanine-rich domain,
which targets TAFII60, and a C-terminal acidic region, which has an
unknown role. Transcriptional activation of a <i>bcd</i> target, the <i>hb</i> promoter, is synergistically
enhanced in vitro by Bcd and Hb. However, this effect is observed only
when both TAFII60 and TAFII110 are present. It has been suggested that
the synergy observed in vivo is because of the corecruitment of
TAFII110 and TAFII60 by Bcd and Hb, respectively. Flies were generated
carrying <i>bcd</i> transgenes lacking one or several of these domains to test their function in vivo. Surprisingly, a <i>bcd</i> transgene that lacks all three putative activation domains is able to rescue the <i>bcdE1</i>
null phenotype to viability. Moreover, the development of these embryos
is not affected by the presence of dominant negative mutations in
TAFII110 or TAFII60. This means that the interactions observed in vitro
between Bcd and TAFII60 or TAFII110 aid transcriptional activation but
are dispensable for normal development (Schaeffer, 1999). </p><p>Bicoid (Bcd), the anterior determinant of Drosophila, controls
embryonic gene expression by transcriptional activation and
translational repression. Both functions require the homeodomain (HD),
which recognizes DNA motifs at target gene enhancers and a specific
sequence interval in the 3' untranslated region of Caudal (<i>CAD</i>)
mRNA. The Bcd HD has been shown to be a nucleic acid-binding unit. Its
helix III contains an arginine-rich motif (ARM), similar to the
RNA-binding domain of the HIV-1 protein REV, needed for both RNA and
DNA recognition. Replacement of arginine 54, within this motif, alters
the RNA but not the DNA binding properties of the HD. Corresponding
BCD mutants fail to repress <i>CAD</i> mRNA translation, whereas the transcriptional target genes are still activated (Niessing, 2000).





</p><p>In order to characterize portions and individual amino acid
residues of the Bcd HD that are specifically required for one or both
Bcd regulatory functions, transgenes expressing wild-type or mutant <i>bcd</i> cDNAs were placed into the genome of homozygous <i>bcd</i> mutant females and their ability to rescue wild-type zygotic <i>hb</i> activation and <i>cad</i> mRNA translation in their embryos was assayed.  Such embryos, referred to as '<i>bcd</i> embryos,' fail to exert Bcd-dependent transcriptional activation of the zygotic target gene <i>hb</i> in their anterior half. Instead, the embryos show a duplication of the posterior Bcd-independent stripe of <i>hb</i> expression in the anterior region  (Niessing, 2000).





</p><p>Expressed Bcd mutant proteins that lack the helices I and II of the HD (Bcd<sup>DeltaH1-2</sup>) or the amino acid interval between positions 42 and 51 in helix III (Bcd<sup>T42-N51</sup>) fail  to restore Bcd-dependent <i>hb</i> transcriptional activation and translational repression of <i>CAD</i> mRNA in the anterior region of <i>bcd</i>
embryos. This indicates that the integrity of the Bcd HD is necessary
for the control of transcription and translation. Transgene-dependent
expression of Bcd<sup>hIIIAntp</sup>, in which the C-terminal half of the Bcd HD is exchanged for the corresponding sequence of the <i>Antennapedia</i> (<i>Antp</i>) HD, rescues Bcd-dependent <i>hb</i> expression in the anterior region of <i>bcd</i>
embryos, but no Cad gradient is formed. Bcd mutations in which two
adjacent arginines at positions 53-54 and 54-55 of the HD,
respectively, were replaced, fail  to control Bcd-dependent
transcription and translation. Thus, helix III of the Bcd HD is
necessary for both transcriptional activation and translational
repression, and amino acids within helix III are essential for
specifying not only DNA binding but also RNA recognition by the HD.
This proposal is consistent with the observation that part of the helix
III of the Bcd HD has characteristics of an arginine-rich motif (ARM)
(Niessing, 2000). </p><p>To test whether the conserved amino acids of Bcd's ARM are
indeed required for RNA target recognition and whether single amino
acid replacements may allow the DNA and RNA binding properties to
separate, alanine replacement mutants of the Bcd HD were generated and
their in vitro binding properties assayed.  The Bcd HD (HD<sup>wt</sup>) binds both DNA and RNA, whereas HD<sup>K50A</sup>, HD<sup>N51A</sup>, HD<sup>R53A</sup>, and HD<sup>R55A</sup> failed to bind to both targets. Bcd HD<sup>R54A</sup>,
which contains alanine in place of arginine in position 54 of the HD,
bound DNA properly, but its RNA binding was reduced by more than one
order of magnitude. The binding properties of HD<sup>K57A</sup> were indistinguishable from HD<sup>wt</sup>.
In summary, arginine at position 54 of the HD is critical for
specifying RNA versus DNA binding, and its replacement shifts the
binding property of the HD to prefer DNA over RNA recognition
(Niessing, 2000). </p><p>In order to test the in vivo relevance of these binding
studies, the corresponding Bcd HD mutants were examined  by
transgene-dependent expression in <i>bcd</i> embryos. The Bcd mutants were generated in the context of an 8.7 kb genomic DNA fragment spanning the entire <i>bcd</i> locus, which fully rescues <i>bcd</i> embryos after P element-mediated transformation. The transgene-expressed Bcd<sup>K57A</sup> protein, which contains an HD with normal DNA and RNA binding properties, causes Bcd-dependent <i>hb</i> expression and Cad gradient formation, and the embryos developed into normal-looking larvae and fertile adults. Bcd<sup>N51A</sup>, Bcd<sup>R53A</sup>, and Bcd<sup>R55A</sup>, which contain HD mutations that cause  the loss of DNA and RNA binding properties in vitro, fail  to activate Bcd-dependent <i>hb</i> transcription and to repress translation of <i>CAD</i> mRNA; such embryos develop  a <i>bcd</i> mutant phenotype. The Bcd<sup>R54A</sup> mutant, which contains an HD with DNA, but no RNA, binding properties, was able to activate the transcription of <i>hb</i> but not to repress the translation of <i>CAD</i> mRNA. This observation is consistent with the result obtained using the transgene bearing the Bcd<sup>R54S</sup>
mutation, which contains a serine residue in place of arginine at
position 54. Thus, both Bcd mutants that contain a replacement of
arginine at position 54 of the HD fail to control <i>CAD</i> mRNA translation but do activate transcription of <i>hb</i> (Niessing, 2000).





</p><p><a href="http://www.sdbonline.org/fly/torstoll/nanos1.htm">Nanos</a> functions as a localized determinant of posterior pattern. <i>Nanos</i>

RNA is localized to the posterior pole of the maturing egg cell.  It encodes a protein that emanates

from this localized source. Nanos acts as a translational inactivator of <i>hunchback</i> and thereby establishes the early anterior to posterior gradient of Hunchback (Curtis, 1995).  





</p><p>Terminal gap genes<i> tailless</i> and<i> huckebein</i> direct the

formation of the posterior Hunchback stripe.  The TLL protein binds in vitro to specific sites

within the 1.4 kb posterior enhancer region, providing the first direct evidence for activation of

gene expression by TLL.  The anterior border of the posterior HB stripe

is determined by TLL concentration in a manner analogous to the activation of anterior <i>hb</i> expression

by Bicoid. In the posterior expression pattern of <i>hb</i>, the transcription factor serves as a (so-called) secondary gap gene regulated by "primary" gap genes <i>tailless</i> and <i>huckebein</i>, regulated in turn by<i> torso</i> (Margolis, 1995). 







</p><p>The Krüppel  binds to the

sequence AAGGGGTTAA.   Binding sites are present for KR upstream of

the two <i>hb</i> promoters.  These could mediate the repression of <i>hb</i> by KR and perhaps

allow <i>hb</i> to influence its own expression.  A 10 Kb genomic DNA fragment contains the <i>hb</i> coding sequence and both  promoters. The proximal promoter directs early zygotic expression of <i>hb</i> in 

the anterior part of the embryo  The distal <i>hb</i>
promoter is transcribed maternally and also directs later zygotic
expression . This latter fragment contains the KR binding sites.  300
bp upstream of the transcription start of the 2.9 kb transcript are
sufficient for normal regulation of the expression of this
transcript.  The two KR binding sites are located at -676 and -359 bp
from the proximal <i>hb</i> promoter (Treisman, 1989).









</p><p>The asymmetric distribution of the gap gene <i><a href="http://www.sdbonline.org/fly/segment/knirps1.htm">knirps</a> (kni)</i> in discrete expression domains is

critical for striped patterns of pair-rule gene expression in the Drosophila embryo. To test

whether these domains function as sources of morphogenetic activity, the stripe 2 enhancer

of the pair-rule gene <i><a href="http://www.sdbonline.org/fly/segment/evenskp1.htm">even-skipped</a> </i> was used to express <i>kni</i> in an ectopic position.

Manipulating the stripe 2-<i>kni</i> expression constructs and examining transgenic lines with

different insertion sites led to the establishment of a series of independent lines that

display consistently different levels and developmental profiles of expression. Individual

lines show specific disruptions in pair-rule patterning that are correlated with the level

and timing of ectopic expression. No effect on the expression patterns of <i>giant</i> or <i>Krüppel</i> could be observed at any level of <i>kni</i> misexpression.  However, the ectopic <i>kni</i> did significantly alter the <i>hunchback</i> pattern.  Stripe 2-<i>kni</i>, centered on PS3, completely prevents the expression of the PS4 <i>hb</i> stripe.  This expression occurs even in embryos that contain the lowest levels of ectopic <i>kni</i> (Kosman, 1997). 







 

</p><p>High local concentrations of NOS

protein in the posterior of the embryo are necessary to inhibit translation of the transcription factor

Hunchback in this region, and thus permit expression of genes required for abdomen formation (Gavis, 1994). 





</p><p>Nanos prevents the repressor <i>hunchback</i> from acting in the posterior half of the embryo. This allows Caudal to activate the gap genes <i>giant </i>and <i>knirps</i> (Rivera-Pomar, 1995). 

</p><p>Mutations in several <a href="http://www.sdbonline.org/fly/aignfam/polycomb.htm">Polycomb group (PcG)</a>
genes cause maternal-effect or zygotic segmentation defects, suggesting
that PcG genes may regulate the segmentation genes of Drosophila.
Individuals doubly heterozygous for mutations in <i><a href="http://www.sdbonline.org/fly/polycomb/polyhome.htm">polyhomeotic</a></i> and six other PcG

genes show gap, pair rule, and segment polarity segmentation defects.

<i><a href="http://www.sdbonline.org/fly/polycomb/postsexc.htm">Posterior sex combs</a> </i>and<i> polyhomeotic</i> interact with<i> Krüppel</i> and

enhance embryonic phenotypes of <i>hunchback</i> and<i> knirps </i> (McKeon, 1994).









</p><p>Genetic experiments and a targeted misexpression approach have been combined to examine the role of the gap gene <i>giant</i> (<i>gt</i>) in patterning anterior regions of the Drosophila embryo. The results suggest that <i>gt</i> functions in the repression of three target genes, the gap genes <i>Kruppel (Kr)</i> and <i>hunchback (hb)</i>, and the pair-rule gene <i>even-skipped (eve)</i>. The anterior border of <i>Kr</i>, which lies 4-5 nucleus diameters posterior to

nuclei that express <i>GT</i>
mRNA, is set by a threshold repression mechanism involving very low
levels of Gt protein.  The gap gene Kr is activated in a broad central
region of precellular embryos. Midway through cleavage cycle 14, this
domain extends from 41-59% egg length. The initial positioning of the
anterior border of this domain is thought to be controlled by
repression involving a combination of maternal and zygotic <i>hunchback</i> transcripts. To test whether <i>gt</i>

is also involved in setting or maintaining this border, the <i>Kr</i> expression pattern was analyzed in embryos containing the st2-gt transgene, a modified version of the 480 bp <i>eve</i> stripe 2

enhancer. These embryos show  no changes in the

initial positioning of the <i>Kr</i> expression domain early in

cleavage cycle 14, but slightly later there is a dramatic

retraction of the anterior <i>Kr</i> border. The delay in the observed repressive effect on the <i>Kr</i> anterior border is probably due to the fact that the <i>Kr</i> domain is expressed earlier

than the st2-gt transgene. Higher levels of ectopic <i>gt</i> result in a more severe retraction, suggesting that <i>Kr</i> transcription is very sensitive to repression by <i>gt</i>. To test whether <i>gt</i> affects <i>Kr</i> expression during normal development, <i>Kr</i> expression was examined in embryos that carry

a strong hypomorphic <i>gt</i> allele. The initial <i>Kr</i> expression

pattern was correctly established in these <i>gt</i> hypomorphic embryos. However, slightly later,  a significant anterior expansion (from 59% to 65% egg length) is observed,

suggesting that <i>gt</i>-mediated repression is essential for

maintaining the position of the anterior border of the <i>Kr</i> domain (Wu, 1998).





</p><p><i>gt</i> is required for repression of zygotic <i>hb</i> expression in more anterior

regions of the embryo. Zygotic expression of <i>hb</i> is initially activated by

the <i>bcd</i> and maternal <i>hb</i> gradients in a broad domain that spans the

anterior half of the embryo. This expression is then rapidly refined

during nuclear division cycle 14, leaving a secondary pattern that

includes a variable head domain, a stripe at the position of

parasegment 4 (PS4), and a posterior stripe. The PS4 stripe overlaps

the anterior border of the Kr domain. By examining

<i>hb</i> expression in <i>gt</i> mutants, significant changes in

this secondary pattern were detected. Initially, <i>hb</i> expression at the

position of PS4 is greatly reduced, possibly because of the

anterior expansion of the <i>Kr</i> domain in <i>gt</i> mutants. High levels of <i>hb</i> expression persist in more anterior regions of <i>gt</i> mutant embryos. The persistent <i>hb</i>

expression domain appears very similar in shape to the

normal <i>gt</i> domain, suggesting that <i>gt</i> may

act as a repressor to clear <i>hb</i> expression from this part of the

embryo during wild-type development. To test whether

endogenous <i>gt</i> levels were required for this repression,  <i>hb</i> expression was examined in <i>gt</i> mutants that also contained the <i>st2-gt</i> transgene. <i>hb</i> expression is repressed normally by a single copy of the <i>st2-gt5</i> transgene, suggesting that relatively low levels of ectopic <i>gt</i> can replace this function of the endogenous gene. Since <i>gt</i> seems to be involved in repression of <i>hb</i> in anterior regions, it is possible that this repression is important for

setting the anterior border of the <i>hb</i> PS4 stripe during wild-type

development. To test this, <i>hb</i> expression was examined in

embryos containing the <i>st2-gt</i> transgene. The position of the anterior border of the <i>hb</i> PS4 stripe appears unchanged in these embryos, suggesting that the levels of

ectopic <i>gt</i> tested here are not sufficient to repress <i>hb</i>
PS4 expression. However, a slight posterior expansion of this stripe
could be detected in embryos with high levels of misexpression, which is
probably caused by the retraction of the Kr domain. This supports the
hypothesis that Kr activity is important for setting the posterior PS4
stripe border, and further demonstrates the importance of <i>gt</i>-mediated restriction of Kr expression to central regions of the embryo (Wu, 1998). 





 





</p><p>Anterior terminal development is controlled by several

zygotic genes that are positively regulated at the anterior

pole of Drosophila blastoderm embryos by the anterior

(<i>bicoid</i>) and the terminal (<i>torso</i>) maternal determinants.

Most Bicoid target genes, however, are first expressed at

syncitial blastoderm as anterior caps, which retract from

the anterior pole upon activation of Torso. To better

understand the interaction between Bicoid and Torso, a

derivative of the Gal4/UAS system was used to selectively

express the best characterized Bicoid target gene,

<i>hunchback</i>, at the anterior pole when its expression should

be repressed by Torso. Persistence of <i>hunchback</i> at the pole

mimics most of the <i>torso</i> phenotype and leads to repression

at early stages of a labral (<i>cap'n'collar</i>) and two foregut

(<i>wingless</i> and <i>hedgehog</i>) determinants that are positively

controlled by <i>bicoid</i> and <i>torso</i>. These results uncovered an

antagonism between <i>hunchback</i> and <i>bicoid</i> at the anterior

pole, whereas the two genes are known to act in concert for

most anterior segmented development. They suggest that

the repression of <i>hunchback</i> by <i>torso</i> is required to prevent

this antagonism and to promote anterior terminal

development, depending mostly on <i>bicoid</i> activity (Janody, 2000).



</p><p>The results indicate that early anterior expression of a labral

determinant, <i>cnc</i>, and of two foregut determinants, <i>wg</i> and <i>hh</i>,

is repressed when zygotic expression of <i>hb</i> is allowed to persist

at the anterior pole of the Drosophila blastoderm embryo.

Expression of <i>cnc, wg</i> and <i>hh</i> is under the positive regulation

of <i>bcd</i> and <i>torso</i> but no zygotic gene has yet been implicated

in this control. This suggests that the Hb protein is able to repress the three genes <i>cnc, wg</i> and <i>hh</i>, and

that <i>torso</i>-induced anterior repression of <i>hb</i> is necessary for

their positive control by <i>torso</i>. To determine whether the

positive control of <i>cnc, wg </i>and <i>hh</i> by <i>torso</i> could be the result

of a double negative control involving <i>hb</i>, expression of these

genes was analysed in <i>hb</i> zygotic mutant embryos derived from

<i>torso</i> females. If the lack of early anterior expression of <i>cnc, wg</i> and <i>hh</i> was solely due to the absence of repression of <i>hb</i>

at the pole, expression of these genes should be recovered in

<i>hb</i> minus embryos derived from <i>torso</i> females. Early anterior expression of <i>cnc, wg</i>  and <i>hh</i> is

not recovered in <i>hb</i> minus embryos derived from <i>torso</i> females

whereas it is normal in <em>hb</em> minus embryos. This indicates

that, although necessary, the anterior repression of <i>hb</i> is not

sufficient to mediate Torso positive control on <i>cnc, wg</i> and <i>hh</i>

early anterior expression  (Janody, 2000).













</p><p>During embryonic development, orderly patterns of gene
expression eventually assign each cell in the embryo its particular
fate. For the anteroposterior axis of the Drosophila embryo, the first
step in this process depends on a spatial gradient of the maternal
morphogen Bicoid (Bcd). Positional information of this gradient is
transmitted to downstream gap genes, each occupying a well defined
spatial domain. The precision of the initial process has been determined
by comparing expression domains in different embryos. The Bcd gradient
displays a high embryo-to-embryo variability, but this noise in the
positional information is strongly decreased ('filtered') at the level
of <i>hunchback</i> (<i>hb</i>) gene expression. In contrast to the Bcd gradient, the <i>hb</i>
expression pattern already includes the information about the scale of
the embryo. Genes known to interact directly with Hb are not
responsible for its spatial precision, but the maternal gene <i>staufen</i> may be

crucial (Houchmandzadeh, 2002).



</p><p>Among all the mutations studied, the only ones that

affect Hb boundary precision are certain alleles of <i>staufen</i>. In embryos from mothers homozygous for either <i>stau<sup>HL</sup> </i>or <i>stau<sup>r9</sup></i>, the Hb boundary position shows a variability of

6%, comparable to the observed Bcd variability.

Surprisingly, this variability is largely reduced (to 2%) in another

strong allele of <i>stau</i>, <i>D3</i>. Mutations in <i>stau</i> disrupt <i>bcd</i> and <i>osk</i> mRNAs and decrease Bcd protein level about twofold. Whether the effect of <i>stau</i> on <i>Hb</i> is simply an indirect effect of its variable effect on <i>bicoid</i> was tested. From the pool of embryos in <i>stau<sup>HL</sup></i> background that were double stained for Bcd and Hb, 

two populations were selected: one that displayed an anterior Hb boundary shift,

and one that displayed a posterior shift. The corresponding

average Bcd profiles for these two populations are very similar, both

in the Bcd level and in its spatial distribution. Thus, the

observed variability in the Hb boundary position may reflect an

activity of <i>staufen</i> independent of <i>bcd</i>. The disruption of Hb

precision in <i>stau<sup>HL</sup></i>

is transmitted to downstream genes, and is

not corrected before gastrulation. For instance, double staining for

Hb and Kr shows that the variability of the Kr

boundary in the <i>stau<sup>HL</sup></i>
background is similar to that of the Hb boundary. Moreover, the
positions of these two boundaries remain tightly correlated, as in the
wild type (Houchmandzadeh, 2002). </p><p>By quantitatively analyzing the protein profiles of maternal
morphogens and zygotic gap genes in numerous wild-type and mutant
embryos, two phenomena  that    take place in the early Drosophila
development have been demonstrated:  (1) at a very early stage, noise
associated with the maternal gradient of Bcd is filtered out, and (2)
at the same time, the genetic network, which includes the Hb gap gene,
establishes spatial proportions (scaling) in the embryo. It is
potentially significant that <i>staufen</i>, the one gene
affecting the process, makes a product that localizes to both poles of
the egg. More work is needed to establish the mechanisms that control
the spatial scaling and precision. It would  then be interesting to
investigate whether similar phenomena are present in other
developmental processes in Drosophila and other organisms
(Houchmandzadeh, 2002). </p><p></p></dd><dt><strong>Overlapping mechanisms function to establish transcriptional quiescence in the embryonic Drosophila germline: Silencing of <i>hunchback</i></strong></dt><dd>







<p>In Drosophila, the germline precursor cells, i.e. pole cells, are
formed at the posterior of the embryo. As observed for newly formed
germ cells in many other eukaryotes, the pole cells are distinguished
from the soma by their transcriptional quiescence. To learn more about
the mechanisms involved in establishing quiescence, a potent
transcriptional activator, Bicoid (Bcd), was ectopically expressed in
pole cells. Bcd overrides the machinery that downregulates
transcription, and activates not only its target gene <i>hunchback</i>
but also the normally female specific Sex-lethal promoter, Sxl-Pe, in
the pole cells of both sexes. Unexpectedly, the terminal pathway gene <i>torso-like</i>
is required for Bcd-dependent transcription. However, terminal signaling
is known to be attenuated in pole cells, and this raises the question
of how this is accomplished. Evidence is presented indicating that <i>polar granule component</i> (<i>pgc</i>) is required to downregulate terminal signaling in early pole cells. Consistently, pole cells compromised for <i>pgc</i> function exhibit elevated levels of activated MAP kinase and premature transcription of the target gene <i>tailless</i> (<i>tll</i>). Furthermore, <i>pgc</i> is required to establish a repressive chromatin architecture in pole cells (Deshpande, 2004).





</p><p>A number of maternally derived gene products are likely to
contribute to transcriptional quiescence in the pole cells of
Drosophila. One of these is Germ cell less (Gcl), a component of the
germ plasm that is necessary for the formation of pole cells. <i>gcl</i> appears to be involved in the establishment of transcriptional quiescence and in embryos lacking <i>gcl</i> activity, newly formed pole buds are unable to silence the transcription of genes such as <i>sisterless-a</i> and <i>scute</i>.
Conversely, when Gcl protein is ectopically expressed in the anterior
of the embryo it can downregulate the transcription of terminal group
genes such as <i>tailless</i> (<i>tll</i>) and <i>huckebein</i>

(Leatherman, 2002). A second maternally derived gene product involved in transcriptional quiescence

is Nanos. In the soma, Nanos, together with Pumilio, plays a key role in

posterior determination by blocking the translation of maternally derived

<i>hunchback</i> (<i>hb</i>) mRNA. Nanos (Nos) also plays a role in down-regulating transcription in pole cells, and in embryos produced by <i>nos</i> mutant mothers:  genes that are normally active only in somatic nuclei are inappropriately transcribed in pole cells. These

include the pair-rule genes <i>fushi tarazu</i> and <i>even

skipped</i>, and the somatic sex determination gene <i>Sex-lethal</i> (Deshpande, 2004 and references therein). 

</p><p>Ectopic expression of Bcd in pole cells can induce the

transcription of the <i>bcd</i> target gene <i>hb</i>. In addition to

activating <i>hb</i>
transcription, Bcd protein perturbs the migration of the pole cells to
the primitive somatic gonad and causes abnormalities in cell cycle
control. These effects on germ cell development resemble those observed
in embryos from <i>nos</i> mutant females. Moreover, as in the case of <i>nos<sup>-</sup></i> pole cells, the <i>Sxl</i> promoter <i>Sxl-Pe</i>
is also turned on in pole cells by Bcd in a sex-nonspecific manner.
Surprisingly, transcriptional activation in pole cells by Bcd requires
the activity of the terminal signaling system. This observation is
unexpected, since previous studies have established that the
transcription of a downstream target gene of the terminal pathway, <i>tailless</i> (<i>tll</i>)
is shut down completely in pole cells. Moreover, the doubly
phosphorylated active isoform of MAP kinase ERK, which serves as a
sensitive readout of the terminal pathway, is nearly absent in pole
cells. Taken together, these findings argue that the activity of
terminal signaling pathway in pole cells of wild-type embryos must be
substantially attenuated, but not shut off completely. What mechanisms
are responsible for downregulating terminal signaling in the
presumptive germline? Evidence indicates that <i>polar granule component</i> (<i>pgc</i>) functions to attenuate the terminal pathway in newly formed pole cells. <i>pgc</i>
encodes a non-translated RNA that is localized in specialized germ
cell-specific structures called polar granules (Nakamura, 1996). Loss
of <i>pgc</i> function in newly formed pole cells results in the
ectopic phosphorylation of ERK and the activation of the ERK dependent
target gene <i>tll</i>. <i>pgc</i> is required to block the establishment of an active chromatin architecture in pole cells  (Deshpande, 2004). 



</p><p>Thus Bcd protein expressed from a

<i>bcd-nos3</i>'<i>UTR</i> transgene (the 3' UTR of <i>nos</i> serves to localize the <i>bcd</i> message to pole cells) can activate the transcription of its target gene <i>hb</i>
in pole cells, overcoming whatever mechanisms are responsible for
transcriptional quiescence. In addition to activating transcription of <i>hb</i>,
Bcd has other phenotypic effects. It prevents the pole cells from
properly arresting their cell cycle and disrupts their migration to
the somatic gonad. Because similar defects in pole cell development
can be induced by the inappropriate expression of Sxl protein in these
cells, one plausible hypothesis is that Bcd not only activates the <i>hb</i> promoter, but also turns on the <i>Sxl</i> establishment

promoter, <i>Sxl-Pe</i>. Consistent with this idea, the <i>Sxl-Pe:lacZ</i>

reporter is turned on in the pole cells of male and female <i>bcd-nos</i>

3' UTR embryos and Sxl protein accumulates in these cells. Although

previous studies indicate that <i>Sxl-Pe</i> is responsive to Bcd, it is somewhat surprising that <i>Sxl-Pe</i>
is not only inappropriately turned on in pole cells by Bcd, but that it
is activated in both sexes. This suggests that Bcd activation of <i>Sxl-Pe</i> in pole cells must proceed by a mechanism that bypasses the X/A chromosome counting system which controls <i>Sxl-Pe</i> activity in the soma. It is interesting to note that the activation of <i>Sxl-Pe</i> in pole cells in the absence of <i>nos</i> function also seems to depend upon a mechanism(s) that circumvents the X/A chromosome counting system (Deshpande, 2004). 



</p><p>That Bcd protein depends upon other ancillary factors to turn on

transcription in pole cells is demonstrated by the requirement for

<i>tsl</i> function in the activation of both the <i>hb</i> and

<i>Sxl-Pe</i> promoters. <i>tsl</i> is a component of the maternal

terminal signaling pathway that activates the zygotic genes, <i>tll</i> and <i>huckebein</i> (<i>hkb</i>), at the poles of the embryo. In addition, the terminal pathway has opposing effects on the expression of

<i>bcd</i>-dependent gap genes. At the anterior pole, where terminal signaling activity is highest, Bcd targets such

as <i>hb</i> and <i>orthodenticle</i> (<i>otd</i>) are repressed. At a distance from the anterior pole, where both the

concentration of Bcd protein and the strength of the terminal signaling

cascade is much lower, the terminal pathway has an opposite, positive effect on <i>hb</i> and <i>otd</i> expression. Two mechanisms are thought to account for the positive effects of the terminal pathway on <i>bcd</i> target genes:  (1) Bcd is a direct target for phosphorylation by the terminal signaling cascade; (2) regulatory regions of <i>bcd</i>
target genes have sites for other transcription factors whose activity
can be directly modulated by the terminal system (Deshpande, 2004).
 </p><p></p></dd><dt><strong>Susceptibility of <i>hunchback</i> to RNAi</strong></dt><dd>









<p>Gene silencing by double-stranded RNA is a widespread phenomenon
called RNAi, involving homology-dependent degradation of mRNAs. RNAi is
established in the Drosophila female germ line. mRNA transcripts are
translationally quiescent at the arrested oocyte stage and are
insensitive to RNAi. Upon oocyte maturation, transcripts that are
translated become sensitive to degradation while untranslated
transcripts remain resistant. Mutations in <i>aubergine</i> and <i>spindleE</i>,
members of the PIWI/PAZ and DE-H helicase gene families, respectively,
block RNAi activation during egg maturation and perturb translation
control during oogenesis, supporting a connection between gene
silencing and translation in the oocyte (Kennerdell, 2002). </p><p>To analyze the effects of dsRNA on mRNA stability in

Drosophila oocytes, dsRNAs corresponding to the

maternally expressed genes <i>bicoid</i> and <i>hunchback</i> were used. These genes were chosen because their mRNAs are synthesized, processed, and

localized to the cytoplasm of oocytes during mid- to late oogenesis. To test the sensitivity of <i>bicoid</i> and <i>hunchback</i> to RNAi, 

fertilized eggs were initially injected with dsRNA. <i>bicoid</i> dsRNA reduces the expression of Bicoid protein and induces a <i>bicoid</i>
loss-of-function phenotype in which embryos have partial transformation
of anterior structures to posterior identities. The effect is robust
enough that dsRNA-coated gold particles randomly introduced into
fertilized eggs by a gene gun generate mutant phenotypes. <i>hunchback</i> dsRNA induces
phenotypes in which embryos are missing thoracic and head segments.
These phenotypes resemble mutant embryos generated when maternal and
zygotic <i>hunchback</i> gene activity is reduced. To determine if
dsRNA injection causes mRNA degradation, endogenous mRNA levels were
measured using a semiquantitative RT-PCR assay. The level of <i>bicoid</i> mRNA was reduced about fourfold 40<sup> </sup>min after injection of <i>bicoid</i> dsRNA. Likewise, injection of <i>hunchback</i> dsRNA resulted in a reduction of <i>hunchback</i> mRNA levels. Coinjection of

a pan-specific ribonuclease inhibitor, vanadyl-ribonucleoside, with

<i>bicoid</i> dsRNA results in no reduction of <i>bicoid</i> mRNA, indicating the effect requires a ribonuclease activity (Kennerdell, 2002).



    

</p><p>Whether and when transcripts become sensitive to dsRNAs during

oogenesis was determined. dsRNA was injected into staged oocytes and their

consequent levels of <i>bicoid</i> and <i>hunchback</i> mRNAs were examined. Although oocytes earlier than stage 14 could not be injected, stage

14 oocytes could be examined for RNAi activity.

Levels of <i>bicoid</i> and <i>hunchback</i> mRNAs were unchanged

in stage 14 oocytes after injection of dsRNA, indicating that oocytes

at this stage are unable to carry out RNAi (Kennerdell, 2002).



</p><p>Oocytes of most animals arrest at species-specific stages of meiosis

while differentiation of the oocytes occurs. Drosophila oocytes arrest transiently in prophase I while the oocytes are loaded

with RNAs and proteins. Some of these

molecules are differentially localized within the oocyte, imparting

positional information to be used for embryonic axis formation. When

Drosophila oocytes reach stage 14, they undergo meiotic arrest

once more, this time at metaphase I. These arrested oocytes remain

translationally quiescent in the ovary, potentially for weeks. Arrest is relieved as in most animal eggs by the process

of maturation or activation that precedes fertilization. In the case of

Drosophila, it appears that ovulation triggers activation of

the oocyte to resume meiosis. When oocytes are

activated, meiosis is completed and translation of maternal RNAs is

dramatically elevated. Shortly thereafter,

the oocyte is fertilized as it passes into the uterus (Kennerdell, 2002).

    

</p><p>RNAi-like effects are not detected in arrested stage 14 oocytes

injected with dsRNA. Was this a general feature of the female germ

line? To explore this issue, dsRNA was injected into mature activated

oocytes. Injection of dsRNA causes reduction in <i>bicoid</i> and

<i>hunchback</i> mRNA levels comparable to those seen in embryos. To
confirm that mRNA sensitivity to dsRNA is strictly coincident with
oocyte maturation, arrested stage 14 oocytes were isolated from
dissected ovaries and the oocytes were activated in vitro. This
maturation procedure reactivates meiosis, mRNA translation, and
vitelline membrane cross-linking. After maturation, oocytes were
injected with <i>bicoid</i> dsRNA and assayed for <i>bicoid</i> mRNA levels. These oocytes showed a

decrease in <i>bicoid</i> mRNA. Thus, immature

Drosophila oocytes that are coordinately blocked for meiosis

and translation are resistant to RNAi, and the block to these processes

can be released by maturation or activation of oocytes (Kennerdell, 2002).

    



</p><p>There are several possible ways in which RNAi might be blocked
in arrested oocytes. One possibility is that an essential component of
the RNAi machinery might be missing at this stage. Oocyte maturation
would then involve synthesis of the component. To address if synthesis
of a missing component is responsible, oocytes were activated in the
presence of the protein synthesis inhibitor cycloheximide. Arrested
stage 14 oocytes were preincubated with cycloheximide and then
activated in vitro in the presence of cycloheximide. This treatment
inhibits &gt;95% of the protein synthesis that occurs during
maturation. These oocytes were injected with <i>bicoid</i>

dsRNA and, strikingly, they showed a decrease in <i>bicoid</i> mRNA

levels that was comparable to that of normal mature oocytes.

RNAi is established during oocyte maturation even when protein

synthesis is blocked. Thus, RNAi establishment during oocyte

maturation does not likely occur by synthesis of an essential

protein component of the RNAi machinery (Kennerdell, 2002).

    

</p><p>The stage 14 oocyte is coordinately blocked in both translation
and RNAi. The two processes are released near simultaneously from this
 block, suggesting perhaps that a shared mechanism links their
regulation. To test this possibility, the effectiveness of dsRNA was
examined against a transcript that is present but not translated after
oocyte maturation. The <i>alphaTubulin67C</i> gene encodes one of
three alpha-tubulin proteins synthesized during oogenesis and
embryogenesis. Transcript accumulates and is actively translated in
early immature oocytes. However, after oocyte maturation, no
translation of <i>alphaTubulin67C</i>

mRNA occurs, even though transcripts at this stage are associated with

ribosomes and are competent to drive translation in vitro. The stable pool of

<i>alphaTubulin67C</i> mRNA is comparable to levels of

<i>bicoid</i> and <i>hunchback</i> mRNA in mature oocytes. When two nonoverlapping dsRNAs against <i>alphaTubulin67C</i>
transcript were independently injected into mature activated oocytes, no
destruction of mRNA was detected. This suggests that the ability of
dsRNAs to destroy transcripts during oogenesis is coupled to the
translation activity of the transcript. Successful translation of
transcripts is perhaps necessary to link a transcript to
dsRNA-triggered degradation (Kennerdell, 2002). </p><p>Several Drosophila genes have been identified that affect

translation of maternal mRNAs during oogenesis. One of these genes, <i>aubergine</i> (<i>aub</i>), encodes a protein with a PIWI and PAZ domain. To determine whether Aub has any role for RNAi in oocytes, the effect of <i>aub</i> mutations on RNAi activity was examined. <i>bicoid</i> and <i>hunchback</i> dsRNAs were injected into <i>aub</i> mutant oocytes that were activated in vitro. Degradation of <i>bicoid</i> and <i>hunchback</i> mRNAs was not observed in <i>aub</i> mutants, indicating that Aub is necessary for germ-line RNAi. Two independent <i>aub</i>
alleles in heteroallelic combination produced the same result,
indicating that the effect was not due to the influence of linked
modifiers (Kennerdell, 2002). </p><p>The <i>aub</i> gene is a member of a family of genes implicated in

RNAi and PTGS. Indeed, <i>aub</i> has been implicated in PTGS

regulation of the <i>Stellate</i> repeats and <i>Su(Ste)</i> genes

on X and Y chromosomes. Another member of the

family, <i>piwi</i>,
has been implicated in PTGS within somatic cells. A third family
member, Ago2, is a subunit of the mRNA-cleaving complex that mediates
RNAi in Drosophila embryonic cells. Thus, several members of this gene
family in Drosophila have been implicated in RNAi and PTGS at various
steps (Kennerdell, 2002). </p><p>It was of interest to determine if other translational regulatory genes

are involved in RNAi. To test this possibility, two genes

that possibly act through interactions with RNA were examined. <i>vasa</i> and

<i>spindle-E</i> (<i>spn-E</i>) encode DexH-box RNA helicases. When activated <i>spn-E</i> mutant oocytes were injected with <i>bicoid</i> or

<i>hunchback</i> dsRNAs, no reduction in cognate mRNA levels occurred. In contrast, activated <i>vasa</i> mutant oocytes injected with <i>bicoid</i>
dsRNA were found to show transcript degradation comparable to wild type.
It is concluded that activation of RNAi in oocytes is dependent on the
activity of Spn-E but not Vasa (Kennerdell, 2002). </p><p>Arrested Drosophila oocytes are unable to generate RNAi
silencing of endogenous maternal mRNAs, but selectively establish this
capability upon egg maturation. How is RNAi activated by egg maturation?
It is argued that RNAi is linked in some way to translation of maternal
mRNAs, which is also specifically activated by egg maturation.
Establishment of RNAi is probably not caused by translation of a
missing RNAi component. Rather, the complete RNAi apparatus may be
present and poised for action but is unable to target homologous
substrate mRNAs until egg maturation. Translational masking of mRNAs, a
mechanism that operates on maternal Drosophila gene expression, may
conceivably be one way in which mRNA is blocked from RNAi attack.
Alternatively, targeting of mRNA might require transcripts be
assembled onto active polysomes. This may be the case, because
siRNA-containing RISC complexes physically fractionate with polysomes,
and siRNAs associate with polysomes in <i>Trypanosoma brucei</i>. There is no

evidence to indicate that dsRNA-targeting requires ribosome

translocation on transcripts, because it is found that cycloheximide

inhibition of ribosome translocation does not block RNAi activity in

activated mature oocytes (Kennerdell, 2002).





</p><p>Coupling RNAi to translated mRNA might facilitate base-pairing

interactions between siRNAs and an unfolded mRNA target, or it might

simply be a means to mark RNAs to be scanned for destruction. The key

evidence suggesting that transcript translation is linked to transcript

degradation by RNAi comes from experiments in which dsRNA

against the <i>alphaTubulin67C</i> message was tested. dsRNA is

ineffective against the untranslated <i>alphaTubulin67C </i>transcript in mature activated oocytes, which are nevertheless competent to carry

out RNAi against translated <i>bicoid</i> and <i>hunchback</i>
transcripts. Thus, there is a correlation between the ability of a
transcript to be translated and its ability to be destroyed by dsRNA
(Kennerdell, 2002). </p><p>Thus Aub and Spn-E are required for RNAi in Drosophila oocytes. Aub and

Spn-E might play a specific role in gene silencing mechanisms,

including RNAi, that nevertheless have a widespread impact on many

features of development. Alternatively, Aub and Spn-E could be required

for RNAi because they activate translation of germ-line transcripts

including those for <i>bicoid</i> and <i>hunchback</i>. Although

there is no evidence for translational control of <i>bicoid</i> mRNA

in <i>aub</i> mutants, these mutants may perturb

steps in the translation of transcripts that are essential for

triggering RNAi. Future experiments should define the specific roles

for Aub and Spn-E in dsRNA-mediated destruction and its relationship to translation control (Kennerdell, 2002).



</p><p></p></dd></dl><hr>

<!--Copy links here--> 

 <i>hunchback</i>:<font size="-1">

<a href="http://www.sdbonline.org/fly/segment/hunchbk1.htm#Bio">Biological Overview</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk2.htm#EvHoms">Evolutionary Homologs</a> 





| <a href="http://www.sdbonline.org/fly/segment/hunchbk7.htm#TofA">Targets of activity</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk4.htm#PROT">Protein Interactions</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk8.htm#POST">Post-transcriptional Regulation</a> 





| <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#DevBio">Developmental Biology</a> 

 | <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#EfMu">Effects of Mutation</a>



 | <a href="http://www.sdbonline.org/fly/segment/hunchbk6.htm#Ref">References</a></font>



<p><a href="http://www.sdbonline.org/fly/aimain/1aahome.htm">Home page</a>: The Inter<i>active</i> Fly © 1995, 1996 Thomas B. Brody, Ph.D.</p><p> Please e-mail comments/corrections to <a href="mailto:brodyt@codon.nih.gov"><em>brodyt@codon.nih.gov</em></a>

<br>

</p><p><a href="http://www.sdbonline.org/"><img src="hunchbk3_files/SDBSmLogo.gif" alt="SDB Logo" hspace="3" vspace="3" height="50" width="50" align="middle">SDB home page</a> || <a href="http://www.sdbonline.org/google.search.html">Search the SDB site </a> 

</p></body></html>