<html><head><title>Interactive Fly, Drosophila</title></head>

<body bgcolor="#ffffff"><font size=""></font>



<p>

<a name="PROT"><i>hunchback</i> 

</a></p><hr>

<a name="PROT"><strong><center> PROTEIN INTERACTIONS</center></strong></a><dl><dt></dt><dd><p>











</p><p><a name="PROT">Drosophila </a><a href="http://www.sdbonline.org/fly/polycomb/mi-2-1.htm">Mi-2</a>
protein binds to a domain in the gap protein Hunchback which is
specifically required for the repression of HOX genes. Using LexA-Hb
as bait,  cDNAs were isolated representing six different genes. In
interaction tests with various unrelated LexA baits, proteins encoded
by three of the six cDNAs interacted exclusively with Hb. Among these
proteins, the <i>hip76</i>
clone product exhibits the strongest interaction with Hb. Multiple cDNA
clones were isolated that span a complete open reading frame (ORF)
encoding a 1982-amino acid protein with high sequence similarity to the
human autoantigen. dMi-2 contains five conserved sequence motifs that
are also present in the two human Mi-2 proteins and in two <i>Caenorhabditis elegans</i> ORFs: two chromodomains, a DNA-stimulated adenosine triphosphatase (ATPase) domain, two

PHD finger motifs, a truncated helix-turn-helix motif resembling the DNA-binding domain of c-myb, and a motif with similarity

to the first two helices of an HMG domain (Kehle, 1998).



</p><p> To map the dMi-2-interacting domain in Hb, Hb fragments were
generated and tested for dMi-2 interaction in yeast two-hybrid assays.
dMi-2 interacts very strongly with sequences overlapping the D domain,
a stretch of amino acids that is conserved between Hb proteins of
different insect species. Mutations in the D box cause extensive
derepression of HOX genes of the Bithorax complex (BXC). Both D box
alleles are premature termination codons, suggesting that the D domain
and its COOH-terminal flanking sequences are critical for repression
of BXC genes. The interaction tests show that this protein portion of
Hb interacts with dMi-2. In vitro binding assays with bacterially
expressed dMi-2 and Hb proteins confirm that these proteins bind
directly to each other. Thus, dMi-2 binds to a portion of Hb that
appears to be critical for repression of BXC genes  (Kehle, 1998).
 </p><p><i>dMi-2</i> homozygotes survive until the first or second larval instar. Mutant embryos and larvae show no obvious mutant phenotypes.

Specifically, expression of BXC genes such as <i>Ultrabithorax </i> (<i>Ubx</i>) and <i>Abdominal-B</i> (<i>Abd-B</i>) is completely normal in these

mutant embryos. This normal expression may be due to maternally deposited <i>dMi-2</i> RNAs or proteins that

persist through subsequent development. Consistent with this,  all early embryos from a dMi-2 deletion stock (including

those lacking the gene) show the same high levels of <i>dMi-2</i> RNA.

An attempt was made to generate embryos from mutant <i>dMi-2</i> germ cells. However, germ cells that are mutant for any of the seven

tested d<i>Mi-2</i> alleles fail to develop. This failure can be rescued by a <i>dMi-2</i> transgene, demonstrating that

dMi-2 is essential for the development of germ cells  (Kehle, 1998). 



</p><p>

An attempt was made to detect a genetic interaction between <i>dMi-2</i> and <i>hb</i>. <i>hb9Q</i> mutants (carrying a premature stop codon upstream of the

first finger domain) show only slight anterior derepression of <i>Ubx</i> in embryos because of perdurance of maternal <i>hb</i> products. <i>hb9K57</i> mutants (carrying a D box lesion) show more extensive anterior derepression of <i>Ubx</i>; this mutant protein is

thought to have dominant-negative effects on the persisting maternal wild-type product. <i>dMi-24;hb9K57</i> double mutants show

much more extensive derepression of <i>Ubx</i> than<i> hb9K57</i> mutants. Similarly, <i>dMi-24;hb9Q</i> double mutants show more

extensive derepression than <i>hb9Q</i> mutants alone. These results demonstrate a synergy between <i>hb</i> and <i>dMi-2</i>
that is consistent with the finding that dMi-2 binds to Hb.
Furthermore, it provides strong evidence that dMi-2 functions in the
repression of BXC genes (Kehle, 1998). </p><p> dMi-2 protein was tested to see if it participates in PcG
repression. As in the case of dMi-2, maternally deposited PcG product
often rescues homozygous mutant PcG embryos to a considerable extent.
Extensive derepression of HOX genes can be observed if such homozygous
embryos are also mutant for another PcG gene. Thus  embryos homozygous
for the PcG gene <i>Posterior sex combs</i> (<i>Psc</i>) and <i>dMi-2</i> were examined and it was found that <i>Ubx</i> and <i>Abd-B</i> are derepressed more extensively in this double mutant than

in <i>Psc</i> homozygotes alone. A similar result was found if <i>dMi-2</i> is combined with other PcG mutations; these

double mutants consistently lead to much enhanced homeotic transformations compared with the single PcG mutants. Thus, there

is a synergy between <i>dMi-2</i> and PcG genes. <i>dMi-2</i> behaves like the PcG mutations <i>Enhancer of Polycomb</i> and <i>Suppressor 2 of zeste</i>,

neither of which on their own cause a homeotic phenotype but do so in combination with other PcG mutations. This suggests that

dMi-2 functions in PcG repression (Kehle, 1998).



</p><p>

Imaginal discs were examined for derepression of HOX genes as well as the phenotypes of their adult derivatives. Clonal analysis

suggests that <i>dMi-2</i> is required for the survival of somatic cells.  Do <i>dMi-2</i> mutations exhibit

gene-dosage interactions with PcG mutations?  While larvae heterozygous for <i>Polycomb (Pc)</i> mutations show  slight derepression of

<i>Ubx</i>, larvae transheterozygous for both <i>Pc</i> and <i>dMi-2</i> mutations show more extensive derepression.

Furthermore, derepression of the HOX gene <i>Sex combs reduced</i> (<i>Scr</i>) in the second and third leg discs of <i>Pc</i>
heterozygotes results in the formation of a first leg structure, the
sex comb, on the second and third legs. The extent of this homeotic
transformation reflects the number of cells that misexpress Scr
protein. This homeotic transformation is far stronger in <i>dMi-2/Pc</i>

transheterozygotes than in adults heterozygous for Pc alone, which is consistent with more extensive derepression of <i>Scr</i>
in the double mutant. These results are further evidence that dMi-2 acts
together with PcG proteins to repress HOX genes (Kehle, 1998). </p><p>It has been proposed that Hb directly or indirectly recruits
PcG proteins to DNA to establish PcG silencing of homeotic genes. The
present data suggest that dMi-2 might function as a link between Hb and
PcG repressors. Although dMi-2 contains two motifs with similarity to
DNA-binding domains (the myb and HMG domains), dMi-2 does not seem to
bind to DNA on its own. Therefore, Hb may recruit dMi-2 to DNA.
Xenopus Mi-2 was recently purified as a subunit of a histone
deacetylase complex with nucleosome remodeling activity. In yeast and
in vertebrates, several transcription factors repress transcription by
recruiting histone deacetylases. It is possible that in Drosophila,
nucleosome remodeling and deacetylase activities of a dMi-2 complex, 
recruited to homeotic genes by Hb, may result in local chromatin
changes that allow binding of PcG proteins to the nucleosomal template.
 Alternatively, the proposed Hb-dMi-2 complex might directly bind a PcG
protein and recruit it to DNA. Finally, the involvement of dMi-2 in
PcG silencing suggests that this process may involve deacetylation of
histones (Kehle, 1998 and references). </p><p> Within the basal

transcription factor complex TFIID, two specific targets, TAFII110 and TAFII60, serve as

coactivators to mediate transcriptional activation by 

BCD and HB. A

quadruple complex containing TATA binding protein (TBP), TAFII250, TAFII110, and TAFII60

mediate transcriptional synergism by BCD and HB, whereas triple TBP-TAFII complexes lacking

one or the other target coactivator fail to support synergistic activation. Deoxyribonuclease I

footprint protection experiments reveal that an integral step leading to transcriptional synergism

involves the recruitment of TBP-TAFII complexes to the promoter by way of multivalent contacts

between activators and selected TAFIIs. Thus, the concerted action of multiple regulators with

different coactivators helps to establish the pattern and level of segmentation gene transcription

during Drosophila development (Sauer, 1996).



</p><p><a href="http://www.sdbonline.org/fly/segment/kruppel1.htm">Krüppel</a> can associate

with the transcription factors encoded by gap genes<i> knirps</i> and <i>hunchback</i>,

affecting Krüppel-dependent gene expression in Drosophila tissue culture cells. The association of

DNA-bound HB protein or free KNI protein with distinct but different regions of Krüppel results in the

formation of DNA-bound transcriptional repressor complexes (Sauer, 1995).





 





</p><p>Translational regulation of <i>Hunchback</i> mRNA is essential for posterior patterning of the Drosophila embryo. This regulation is

mediated by sequences in the 3'-untranslated region of <i>HB</i> mRNA (the Nanos response elements or NREs), as well as two trans-acting

factors --  Nanos and Pumilio.  Pum

binds to a pair of 32-nucleotide sequences (named <u>N</u>anos

<u>r</u>esponse <u>e</u>lements -- NREs) in the 3'-UTR of

maternal <i>HB</i> mRNA in order to repress <i>HB</i>

translation in the posterior of the embryo. This translational repression is essential

for normal abdominal segmentation. The RNA-binding domain of Pum is structurally similar to

that of another translational regulator, FBF (<i>fem-3</i>

mRNA-binding factor) found in <i>C. elegans</i> (Zhang, 1997).

The minimal RNA-binding domain of each protein consists of eight

imperfect repeats plus flanking residues. These structural similarities

define a conserved 'Puf' motif (<u>Pu</u>m and <u>F</u>BF)
that is found in proteins from diverse organisms from yeast to humans.
However, the RNA partner of no other Puf domain protein has been
identified, nor is it clear whether other Puf proteins regulate
translation or some other aspect of RNA metabolism. Thus, Pumilio
recognizes the NREs via a conserved binding motif. The mechanism of
Nanos action has not been clear. In this report protein-protein and
protein-RNA interaction assays in yeast and in vitro were used to show
that Nanos forms a ternary complex with the RNA-binding domain of
Pumilio and the NRE. Mutant forms of the NRE, Nos, and Pum that do not
regulate <i>HB</i>
mRNA normally in embryos do not assemble normally into a ternary
complex. In particular, recruitment of Nos is dependent on bases in the
center of the NRE, on the carboxy-terminal Cys/His domain of Nos, and
on residues in the eighth repeat of the Pum RNA-binding domain. These
residues differ in a closely related human protein that also binds to
the NRE but cannot recruit Drosophila Nos. Taken together, these
findings suggest models for how Nos and Pum collaboratively target <i>HB</i>
mRNA. More generally, they suggest that Pum-like proteins from other
species may also act by recruiting cofactors to regulate translation
(Sonoda, 1999). </p><p>In one model, Nos simultaneously makes specific contacts with
Pum and nucleotides 17-20 of the NRE. On their own, neither the Nos-Pum
nor the Nos-NRE contacts are strong enough to recruit Nos to <i>HB</i> mRNA (at least
in the presence of competitor proteins and RNAs), because binary
complexes with Nos are not detectable. In another model, unbound Pum
cannot interact with Nos, but binding to the NRE induces a
conformational change in Pum, which subsequently recruits Nos via
protein-protein contacts. In this model, nucleotides 17-20 of the NRE
interact with Pum to induce the conformational change without affecting
its affinity for the RNA, and nonspecific interactions between Nos and
other portions of the RNA help stabilize the complex. Either model is
consistent with the nonspecific RNA-binding activity reported for the
carboxy-terminal portion of Nos in vitro and the RNA-Nos cross-link
found in this study. Further structural and biochemical experiments
will be required to distinguish between these (or alternative) models
(Sonoda, 1999 and references therein). </p><p>The mechanism by which the ternary complex blocks translation
is not yet clear. mRNAs subject to Nos- and Pum-dependent repression
are deadenylated in vivo. In addition, Nos and Pum have been shown to
regulate internal ribosome entry site (IRES)-dependent translation in
imaginal disc cells, suggesting that their regulatory target lies
downstream of cap recognition and scanning. It is assumed that some
surface of the ternary complex, formed jointly by Nos and Pum, targets
a component of the polyadenylation or translation machinery. This
surface appears to be altered in the Pum680 mutant protein, which binds
the NRE normally but is defective in regulating <i>HB</i>
translation in the embryo. The Pum680 mutant recruits Nos into a ternary
complex normally  and thus apparently is defective in a subsequent step
of the repression reaction. The RNA-binding domain of Pum therefore
appears to have at least three different functions in regulating <i>HB</i>: recognizing the NRE,

recruiting Nos, and acting as a corepressor (with Nos) to block translation (Sonoda, 1999 and references therein). 



</p><p>In the experiments reported in this study,   focus was placed on
discrete regions of both Nos (the carboxy-terminal 97 amino acids) and
Pum (the minimal RNA-binding domain), which play an essential role in
formation of the ternary complex. However, other regions of Nos are
known to be required for its function in repressing translation in the
embryo. In addition, residues elsewhere in Pum play an unknown role in
augmenting the intrinsic translational repression activity of the
RNA-binding domain. Thus, the ternary complex formed by the 157-kD,
full-length Pum protein may be stabilized by auxillary protein-protein
or protein-RNA interactions in addition to those that mediate
recruitment of the carboxy-terminal domain of Nos by the RNA-binding
(or Puf domain) of Pum. The results  suggest that Puf domain proteins
generally may act by recruiting cofactors to specific RNA binding
sites. Cofactor specificity may be mediated, at least in part, by the
eighth repeat of the Puf domain. Although Puf domain proteins have been
described in organisms from yeast to humans, for only one protein
other than Drosophila Pum, <i>C. elegans</i> FBF, is the relevant RNA regulatory target known. FBF regulates the sperm/oocyte switch in the

hermaphrodite germ line by governing the translation of fem-3 mRNA (Zhang, 1997). The FBF

RNA-binding domain interacts with one of the <i>C. elegans</i> Nos homologs (Kraemer, 1999). Further experiments will be required to determine whether

the Pum/fly Nos complex and the FBF/worm Nos complex function in a similar manner (Sonoda, 1999 and references therein). 







</p><p></p></dd><dt><strong>Selective dimerization of Hunchback and Ikaros</strong></dt><dd>





<p>McCarty, A. S., et al. (2003). Selective dimerization of a C2H2 zinc finger subfamily. Mol. Cell 11: 459-470. Medline abstract: <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12620233&amp;dopt=Abstract">12620233</a> 



</p><p>The C2H2 zinc finger is the most prevalent protein motif in the
mammalian proteome. Two C2H2 fingers in Ikaros are dedicated to
homotypic interactions between family members. These fingers comprise a
bona fide dimerization domain. Dimerization is highly selective,
however, since homologous domains from the TRPS-1 and Drosophila
Hunchback proteins support homodimerization, but not heterodimerization
with Ikaros. Ikaros-Hunchback selectivity is determined by 11 residues
concentrated within the alpha-helical regions typically involved in
base recognition. Preferential homodimerization of one chimeric protein
predicts a parallel dimer interface and establishes the feasibility of
creating novel dimer specificities. These results demonstrate that the
C2H2 motif provides a versatile platform for both sequence-specific
protein-nucleic acid interactions and highly specific dimerization
(McCarty, 2003). </p><p>The consensus sequence for C2H2 zinc fingers is (F/Y)-X-C-X<sub>2-5</sub>-C-X<sub>3</sub>-(F/Y)-X<sub>5</sub>-psi-X<sub>2</sub>-H-X<sub>3-5</sub>-H,
where X is any amino acid and psi is a hydrophobic residue. In addition
to the cysteines and histidines that coordinate zinc, C2H2 fingers
contain conserved hydrophobic residues that pack in the hydrophobic
core. These conserved amino acids lead to the formation of the
characteristic structure comprised of a two-stranded antiparallel ß
sheet and an alpha helix. Structures of several C2H2 domains bound to
DNA have been solved by X-ray crystallography and NMR. The structures
reveal that nucleotide base contacts are mediated primarily by residues
near the N-terminal half of the alpha helix. The four residues most
commonly involved in specific base recognition are at positions -1, 2,
3, and 6, relative to the beginnning of the alpha helix (McCarty,
2003). </p><p>Although most C2H2 fingers appear to contribute to protein-DNA
and protein-RNA interactions, C2H2 fingers have also been implicated in
protein-protein interactions. However, structural information has been
obtained only for domains in which the contribution of the C2H2 finger
to the protein-protein interaction is indirect. For example,
dimerization of the RAG1 recombinase is mediated by a domain containing
a RING finger and a C2H2 zinc finger. In this domain, the C2H2 finger
helps form a stable scaffold upon which the dimer interface is formed,
but the finger does not directly participate in dimerization (McCarty,
2003). </p><p>Ikaros is a protein expressed in hematopoietic cells that has
been implicated in gene silencing and activation. Four C2H2 fingers
near the N terminus of Ikaros are involved in sequence-specific DNA
binding. The C terminus contains two additional C2H2 zinc fingers that
play no apparent role in the protein-DNA interaction. Rather, previous
studies using yeast two-hybrid screens and coimmunoprecipitation assays
have demonstrated that they are essential for self-interactions and for
interactions with the corresponding zinc fingers of other Ikaros family
members. A protein fragment spanning the two zinc fingers was
sufficient for the interaction, which was disrupted by mutations in the
zinc-coordinating cysteines and histidines and by zinc displacement
(McCarty, 2003). </p><p>Although the C-terminal fingers of Ikaros do not directly
contribute to protein-DNA interactions, they are critical for
high-affinity DNA binding, which usually involves the recognition of
tandem binding sites by two subunits of an Ikaros complex. Homotypic
interactions mediated by the C-terminal fingers are also necessary for
the targeting of Ikaros to pericentromeric heterochromatin, which has
been hypothesized to be important for the pericentromeric recruitment
and heritable silencing of Ikaros target genes. Although the C-terminal
fingers are involved in protein-protein interactions, rather than
protein-DNA interactions, both fingers match perfectly the C2H2
consensus sequence, with the exception of a single missing F/Y in the
second finger. The absence of this F/Y does not explain its unusual
function, however, since a hydrophobic residue is missing at this
position in several DNA binding zinc fingers, including the first and
third fingers of the DNA binding domain of Ikaros (McCarty, 2003). </p><p>The zinc finger organization found in mammalian Ikaros
proteins also exists within the Drosophila gap segmentation protein
Hunchback. The C-terminal fingers of Ikaros and Hunchback exhibit
considerable sequence homology, suggesting that the Hunchback fingers
may support dimer formation. To examine this possibility, the Hunchback
C-terminal fingers were substituted for the Ikaros <strong>d</strong>imerization <strong>z</strong>inc <strong>f</strong>inger
(DZF) domain in the context of both the epitope-tagged protein
(f-HbDZF) and untagged IK I (Hb I). Hb I efficiently
coimmunoprecipitates with f-HbDZF, confirming that Hunchback C-terminal
fingers contain a functional DZF domain. Interestingly, Hb I did not
coimmunoprecipitate with f-IkDZF, and IK I did not coimmunoprecipitate
with f-HbDZF. These results demonstrate that DZF-mediated dimerization
is selective. This surprising degree of selectivity was equally
apparent in chemical crosslinking experiments (McCarty, 2003). </p><p>To identify the amino acid determinants of selective
dimerization, the Ikaros and Hunchback DZF sequences were first
compared. This comparison revealed striking similarities and
differences. Within the 60 amino acid region, 23 of the amino acids
(38%) are identical, consistent with the conserved functions of the
domains. The atypical length of the spacer between the final two
histidines (5 residues) is also conserved. Despite the high degree of
identity, five positions contain charge reversals, ten positions vary
between charged and nonpolar residues, and seven positions vary between
polar and nonpolar residues. These three categories represent 37% of
the DZF amino acids, making it difficult to predict which residues are
responsible for dimerization selectivity. Comparison of the Ikaros and
TRPS 1 DZF sequences, or the Hunchback and TRPS 1 sequences, reveals
similar degrees of identity and divergence (McCarty, 2003). </p><p>Because of the high degree of divergence, a systematic
approach was necessary to identify the determinants of selective
dimerization. As a starting point, two series of Ikaros-Hunchback DZF
chimeras were generated in the context of untagged IK. These chimeras
were expressed in HEK 293 cells along with either f-HbDZF or f-IkDZF.
Protein-protein interactions were monitored using a
coimmunoprecipitation assay (McCarty, 2003). </p><p>The first series of chimeras contains decreasing amounts of
the Hunchback DZF N terminus fused to increasing amounts of the Ikaros
DZF C terminus (Hb-IK1 to Hb-IK6). An efficient interaction with
f-HbDZF was retained when Ikaros amino acids 53-64 were included in the
chimeric protein, but not when amino acids 47-64 were included. These
results suggest that at least 1 residue between amino acids 47 and 52
is required for selective homodimerization of the Hunchback DZF, but
that amino acids 53-64 are not involved in selectivity (McCarty, 2003).
 </p><p>The second series of chimeric proteins was used to define the
amino acids at the N terminus of the Hunchback DZF that are required
for selective dimerization. This series contains decreasing amounts of
the Ikaros DZF N terminus fused to increasing amounts of the Hunchback
DZF C terminus (Ik-Hb1 to Ik-Hb6. An efficient interaction with the
f-HbDZF protein was observed when Ikaros amino acids 1-14 were included
in the chimeric protein, but the interaction was lost when Ikaros amino
acids 1-27 were included. These results demonstrate that at least 1
residue between amino acids 15 and 27 is required for selectivity and
that amino acids 1-14 are not involved. Thus, the residues involved in
selective dimerization of the Hunchback DZF are located between amino
acids 15 and 52  (McCarty, 2003). </p><p>The two sets of chimeric proteins were then used to define the
region required for dimerization with the Ikaros DZF. Using the Hb-Ik
series, an efficient interaction with f-IkDZF was observed when the
chimeric protein included Hunchback amino acids 1-27. However, an
interaction was not observed when Hunchback amino acids 1-31 were
included. Using the Ik-Hb series, an efficient interaction with the
f-IkDZF protein was observed when the chimeric protein included
Hunchback amino acids 52-60. Inefficient coimmunoprecipitation was
observed when the chimeric protein included Hunchback amino acids
34-60, and no coimmunoprecipitation was observed when Hunchback amino
acids 32-60 were included. These results suggest that the residues
contributing to selective dimerization of the Ikaros DZF are located
between amino acids 28 and 51 (McCarty, 2003). </p><p>Using a homology model of the Ikaros DZF, it is noted that 7
of the 8 residues identified in this manuscript as important for
selective dimerization cluster on the surface of the model. These
patches are likely to represent the dimer interface. The clusters
correspond primarily to the alpha helices of both zinc finger domains.
Within the first finger, the clustering of the 3 critical residues,
L17, M21, and C28, was expected on the basis of the 7 residue spacing
between M21 and C28 on the alpha helix, and the 4 residue spacing
between M21 and L17. Because the orientation of the alpha helix is
constrained by the conserved hydrophobic residue, Y22, which packs in
the hydrophobic core, there is a high probability that these critical
residues will be exposed in the DZF structure (McCarty, 2003). </p><p>In the second finger, the 4 residues that are most important
for selective dimerization, R47, Y48, E49, and F50, are adjacent to one
another on the alpha helix. A hydrophobic residue that packs in the
core is expected at the position occupied by F50. Therefore, this
residue is almost certainly at the core and does not directly
contribute to the dimer interface. Instead, F50 is likely to make an
important contribution to the structure of the finger. Interestingly,
this residue cannot be replaced by the leucine found at the same
position in the Hunchback DZF. This observation suggests that the
structures of the hydrophobic cores of the Ikaros and Hunchback DZFs
differ, which may be important for selectivity (McCarty, 2003). </p><p>Residues R47, Y48, and E49 are likely to be important
constituents of the dimer interface. Because the hydrophobic cores of
C2H2 motifs are highly compact, all three of these residues are likely
to be exposed. However, the precise orientations of the amino acid side
chains cannot be determined in the absence of an experimentally defined
structure. The presence of arginine, tyrosine, and glutamate residues
at the Ikaros dimerization interface (R47, Y48, and E49) suggests that
a combination of salt bridges, hydrogen bonds, and hydrophobic
interactions stabilize the dimer. Equivalent positions of the Hunchback
DZF sequence contain hydrophobic residues, suggesting that
stabilization of the Hunchback dimerization interface may be more
dependent on van der Waals interactions (McCarty, 2003). </p><p></p></dd></dl><hr>

<!--Copy links here--> 



 <i>hunchback</i>:<font size="-1">

<a href="http://www.sdbonline.org/fly/segment/hunchbk1.htm#Bio">Biological Overview</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk2.htm#EvHoms">Evolutionary Homologs</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk3.htm#Regul">Regulation</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk7.htm#TofA">Targets of activity</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk8.htm#POST">Post-transcriptional Regulation</a> 





| <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#DevBio">Developmental Biology</a> 

 | <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#EfMu">Effects of Mutation</a>



 | <a href="http://www.sdbonline.org/fly/segment/hunchbk6.htm#Ref">References</a></font>





<p></p>

<p><a href="http://www.sdbonline.org/fly/aimain/1aahome.htm">Home page</a>: The Inter<i>active</i> Fly © 1995, 1996 Thomas B. Brody, Ph.D.</p><p> Please e-mail comments/corrections to <a href="mailto:brodyt@codon.nih.gov"><em>brodyt@codon.nih.gov</em></a>

<br>



</p><p><a href="http://www.sdbonline.org/"><img src="hunchbk4_files/SDBSmLogo.gif" alt="SDB Logo" hspace="3" vspace="3" height="50" width="50" align="middle">SDB home page</a> || <a href="http://www.sdbonline.org/google.search.html">Search the SDB site </a> 

</p></body></html>