<html><head><title>Interactive Fly, Drosophila</title></head>

<body bgcolor="#ffffff"><font size=""></font>



<p>

<i>hunchback</i>



</p><hr>







<p><strong><center> REGULATION</center></strong></p><p></p><dl><a name="POST"> 

</a><p></p><dt><a name="POST"> <strong> Posttranscription regulation</strong></a></dt><dd><p>







</p><p><a name="POST">In Drosophila embryos, graded activity of the posterior determinant </a><a href="http://www.sdbonline.org/fly/torstoll/nanos1.htm">Nanos (nos)</a> generates abdominal

segmentation by blocking protein expression from maternal transcripts of <i>hb</i>.
When active inappropriately at the anterior pole, NOS can also block
expression of the anterior determinant Bicoid. Both regulatory
interactions are mediated by similar sequences in the 3' untranslated
region of each transcript. These NOS response elements (NREs) are both
necessary and sufficient to confer nos-dependent regulation, the
degree of regulation determined by the number and quality of the
elements and the level of NOS in vivo. Based on these and other
results, it can be argued that NOS acts as a morphogen, controlling HB
expression (and hence abdominal pattern) as a function of its
concentration-dependent interaction with the NREs.  Thus, it would seem
that the requirements for <i>NOS</i> mRNA localization, involving proteins of the <a href="http://www.sdbonline.org/fly/aignfam/postgrp.htm">posterior group</a> are imposed by the presence of the NRE in BCD and HB mRNAs (Wharton, 1991). 















</p><p>The Drosophila Nanos protein is a localized repressor of <i>Hunchback</i> mRNA translation in the early embryo, and is required for the establishment of the anterior-posterior body axis. Analysis of <i>nanos</i>
mutants reveals that a small, evolutionarily conserved, C-terminal
region      is essential for Nanos function in vivo, while no other
single portion of the Nanos protein is absolutely required. Within the
C-terminal region      are two unusual Cys-Cys-His-Cys (CCHC) motifs
that are potential zinc-binding sites.  One equivalent of zinc is bound
with high affinity by each of the   CCHC motifs. <i>nanos </i>mutations
disrupting metal binding at either of these      two sites in vitro
abolish Nanos translational repression activity in vivo. Full-length
and C-terminal Nanos proteins bind to RNA in vitro with high affinity,
but with little sequence specificity. Mutations      affecting the <i>Hunchback</i> mRNA target sites for
Nanos-dependent translational repression are found to disrupt
translational repression in      vivo, but have little effect on Nanos
RNA binding in vitro. Thus, the Nanos zinc domain does not specifically
recognize target <i>Hunchback</i>

     RNA sequences, but might interact with RNA in the context of a larger ribonucleoprotein complex (Curtis, 1997).





</p><p>Posterior patterning in Drosophila embryos is governed by Nanos repression of

the translation of maternal transcripts of the <i>hunchback</i> gene. Sites in <i>HB</i>
mRNA that mediate this repression, named Nanos response elements
(NREs), have been identified. Two proteins present in embryonic
extracts, neither one NOS, bind specifically to the NRE in vitro. 
Binding in vitro correlates with NRE function in vivo. One of the
NRE-binding factors is encoded by <i><a href="http://www.sdbonline.org/fly/gene/pumilio.htm">pumilio</a></i>, like <i>nos</i>, a gene  essential for

abdominal segmentation. This suggests that PUM acts by recognizing the

NRE and then recruiting NOS. Presumably, the resulting complex inhibits some component of the

translation machinery (Murata, 1995). 









</p><p>Nanos protein promotes abdominal structures in Drosophila embryos by repressing the

translation of maternal <i>hunchback</i> mRNA in the posterior. To study the mechanism of

<i>nanos</i>-mediated translational repression,  the mechanism by which

maternal <i>Hunchback</i> mRNA is translationally activated was examined. In the oocyte from wild-type females, where no <i>HB</i>
translation is detected, the mRNA has a poly(A) tail length of
approximately 30 nucleotides.  However, concomitant with translation of
the mRNA at between 0.5 and 1.5 hours after egg deposition, the poly(A)
tail is elongated to approximately 70 nt.  In the absence of <i>nanos</i>

activity, the poly(A) tail of <i>Hunchback</i> mRNA is elongated to approximately 100 nt concomitant with its

translation, suggesting that cytoplasmic polyadenylation directs activation. However, in

the presence of <i>nanos</i> the length of the <i>Hunchback</i> mRNA poly(A) tail is reduced via the nanos response element present in the <i>HB</i> 3'UTR. To

determine if <i>nanos</i> activity represses translation by altering the polyadenylation state

of <i>Hunchback</i> mRNA,  various in vitro transcribed RNAs were injected into Drosophila

embryos and  changes in polyadenylation were determined. <i>nanos</i> activity reduces the

polyadenylation status of injected <i>Hunchback</i> RNAs by accelerating their

deadenylation. Pumilio activity, which is necessary to repress the translation of

<i>Hunchback</i> mRNA, is also needed to alter polyadenylation. An examination of translation

indicates a strong correlation between poly(A) shortening and suppression of

translation. These data indicate that <i>nanos</i> and <i>pumilio</i> determine posterior morphology

by promoting the de-adenylation of maternal <i>Hunchback</i> mRNA, thereby repressing its

translation (Wreden, 1997). 





</p><p>Posterior patterning in the Drosophila embryo requires the action of Nanos and Pumilio,

which collaborate to regulate the translation of maternal <i>Hunchback</i> mRNA. Pum recognizes sites in the 3' UTR of <i>HB</i> mRNA. The

RNA-binding domain of Pum has been defined and residues essential for translational repression are shown to be

embedded within this domain. Nos and Pum can repress cap-independent

translation from an internal ribosome entry site (IRES) in vivo, suggesting that they act downstream of

the initial steps of normal, cap-dependent translation (Wharton, 1998). 





</p><p>Translation of <i>Hunchback(mat) (HB[mat])</i> mRNA must be repressed in the posterior of the

pre-blastoderm Drosophila embryo to permit formation of abdominal segments. This translational

repression requires two copies of the Nanos Response Element (NRE), a 16-nt sequence in the

<i>HB[mat]</i> 3' untranslated region. Translational repression also requires the action of two proteins: Pumilio

(PUM), a sequence-specific RNA-binding protein; and Nanos, a protein that determines the location of

repression. Binding of Pum to the NRE is thought to target <i>HB(mat)</i> mRNA for repression. The RNA-binding domain of Pum is an evolutionarily conserved, 334 amino acid region at the

carboxy terminus of the approximately 158 kDa Pum protein. This contiguous region of Pum retains

the RNA binding specificity of full length Pum protein. Proteins with sequences homologous to the

Pum RNA binding domain are found in animals, plants, and fungi. The high degree of sequence

conservation of the Pum RNA binding domain in other far flung species suggests that the domain is

an ancient protein motif, and conservation of sequence reflects conservation of function:

that is, the homologous region from a human protein binds RNA with sequence specificity related to

but distinct from Drosophila Pum (Zamore, 1997). 







</p><p>Translational repression of <i>Hunchback</i> mRNA in the posterior of the Drosophila embryo requires

two copies of a bipartite sequence, the Nanos Response Element (NRE), located in the 3' untranslated

region of the mRNA. The Pumilio protein is thought to bind the NREs and thereby repress

<i>HB</i> translation. The RNA-binding domain of Pum defines an evolutionarily conserved family of

RNA-binding proteins: the Pum-Homology Domain (Pum-HD) proteins, which have been identified

in yeast, plants, and animals. The Pum RNA-binding domain [the Drosophila PUM-HD

(DmPUM-HD)] has been shown to recognize nucleotides in both the 5' and 3' halves of the

NRE, suggesting that a dimer of Pum might recognize one NRE. The RNA-binding

affinity and stoichiometry of the DmPUM-HD has been analyzed and it is found that one DmPUM-HD monomer binds

independently and with equal affinity to each NRE (KD approximately 0.5 nM). No

cooperative interactions is detected between DmPUM-HD monomers bound at adjacent sites. These results imply

that a single DmPUM-HD protein recognizes nucleotides in both the 5' and 3' NRE half-sites. Based

on an estimate of the intraembryonic concentration of PUM (&gt;40 nM), it is proposed that in vivo nearly

all NREs are occupied by a PUM monomer (Zamore, 1999). 





</p><p>Translational regulation of <i>Hunchback</i> mRNA is essential for posterior patterning of the Drosophila embryo. This regulation is

mediated by sequences in the 3'-untranslated region of <i>HB</i> mRNA (the Nanos response elements or NREs), as well as two trans-acting

factors --  Nanos and Pumilio.  Pum

binds to a pair of 32-nucleotide sequences (named <u>N</u>anos

<u>r</u>esponse <u>e</u>lements -- NREs) in the 3'-UTR of

maternal <i>HB</i> mRNA in order to repress <i>HB</i>

translation in the posterior of the embryo. This translational repression is essential

for normal abdominal segmentation. The RNA-binding domain of Pum is structurally similar to

that of another translational regulator, FBF (<i>fem-3</i>

mRNA-binding factor) found in <i>C. elegans</i> (Zhang, 1997).

The minimal RNA-binding domain of each protein consists of eight

imperfect repeats plus flanking residues. These structural similarities

define a conserved 'Puf' motif (<u>Pu</u>m and <u>F</u>BF)
that is found in proteins from diverse organisms from yeast to humans.
However, the RNA partner of no other Puf domain protein has been
identified, nor is it clear whether other Puf proteins regulate
translation or some other aspect of RNA metabolism. Thus, Pumilio
recognizes the NREs via a conserved binding motif. The mechanism of
Nanos action has not been clear. In this report protein-protein and
protein-RNA interaction assays in yeast and in vitro were used to show
that Nanos forms a ternary complex with the RNA-binding domain of
Pumilio and the NRE. Mutant forms of the NRE, Nos, and Pum that do not
regulate <i>HB</i>
mRNA normally in embryos do not assemble normally into a ternary
complex. In particular, recruitment of Nos is dependent on bases in the
center of the NRE, on the carboxy-terminal Cys/His domain of Nos, and
on residues in the eighth repeat of the Pum RNA-binding domain. These
residues differ in a closely related human protein that also binds to
the NRE but cannot recruit Drosophila Nos. Taken together, these
findings suggest models for how Nos and Pum collaboratively target <i>HB</i>
mRNA. More generally, they suggest that Pum-like proteins from other
species may also act by recruiting cofactors to regulate translation
(Sonoda, 1999). </p><p>In one model, Nos simultaneously makes specific contacts with
Pum and nucleotides 17-20 of the NRE. On their own, neither the Nos-Pum
nor the Nos-NRE contacts are strong enough to recruit Nos to <i>HB</i> mRNA (at least
in the presence of competitor proteins and RNAs), because binary
complexes with Nos are not detectable. In another model, unbound Pum
cannot interact with Nos, but binding to the NRE induces a
conformational change in Pum, which subsequently recruits Nos via
protein-protein contacts. In this model, nucleotides 17-20 of the NRE
interact with Pum to induce the conformational change without affecting
its affinity for the RNA, and nonspecific interactions between Nos and
other portions of the RNA help stabilize the complex. Either model is
consistent with the nonspecific RNA-binding activity reported for the
carboxy-terminal portion of Nos in vitro and the RNA-Nos cross-link
found in this study. Further structural and biochemical experiments
will be required to distinguish between these (or alternative) models
(Sonoda, 1999 and references therein). </p><p>The mechanism by which the ternary complex blocks translation
is not yet clear. mRNAs subject to Nos- and Pum-dependent repression
are deadenylated in vivo. In addition, Nos and Pum have been shown to
regulate internal ribosome entry site (IRES)-dependent translation in
imaginal disc cells, suggesting that their regulatory target lies
downstream of cap recognition and scanning. It is assumed that some
surface of the ternary complex, formed jointly by Nos and Pum, targets
a component of the polyadenylation or translation machinery. This
surface appears to be altered in the Pum680 mutant protein, which binds
the NRE normally but is defective in regulating <i>HB</i>
translation in the embryo. The Pum680 mutant recruits Nos into a ternary
complex normally  and thus apparently is defective in a subsequent step
of the repression reaction. The RNA-binding domain of Pum therefore
appears to have at least three different functions in regulating <i>HB</i>: recognizing the NRE,

recruiting Nos, and acting as a corepressor (with Nos) to block translation (Sonoda, 1999 and references therein). 



</p><p>In the experiments reported in this study,   focus was placed on
discrete regions of both Nos (the carboxy-terminal 97 amino acids) and
Pum (the minimal RNA-binding domain), which play an essential role in
formation of the ternary complex. However, other regions of Nos are
known to be required for its function in repressing translation in the
embryo. In addition, residues elsewhere in Pum play an unknown role in
augmenting the intrinsic translational repression activity of the
RNA-binding domain. Thus, the ternary complex formed by the 157-kD,
full-length Pum protein may be stabilized by auxillary protein-protein
or protein-RNA interactions in addition to those that mediate
recruitment of the carboxy-terminal domain of Nos by the RNA-binding
(or Puf domain) of Pum. The results  suggest that Puf domain proteins
generally may act by recruiting cofactors to specific RNA binding
sites. Cofactor specificity may be mediated, at least in part, by the
eighth repeat of the Puf domain. Although Puf domain proteins have been
described in organisms from yeast to humans, for only one protein
other than Drosophila Pum, <i>C. elegans</i> FBF, is the relevant RNA regulatory target known. FBF regulates the sperm/oocyte switch in the

hermaphrodite germ line by governing the translation of fem-3 mRNA (Zhang, 1997). The FBF

RNA-binding domain interacts with one of the <i>C. elegans</i> Nos homologs (Kraemer, 1999). Further experiments will be required to determine whether

the Pum/fly Nos complex and the FBF/worm Nos complex function in a similar manner (Sonoda, 1999 and references therein). 





</p><p>Maternally derived <i>HB</i> mRNA is uniformly distributed
throughout the embryo; the mRNA is translationally repressed in the
posterior, giving rise to an anterior-to-posterior gradient of Hb
protein. Failure of this repression results in the abnormal
accumulation of Hb in the posterior, which inhibits abdominal
segmentation.  Two conserved RNA-binding proteins, Pumilio (Pum) and
Nanos (Nos), are specifically required to repress <i>HB</i>
translation. Pum, which is distributed uniformly throughout the embryo,
is the founding member of a large family of RNA-binding proteins. Pum
binds to 32 nucleotide sites in the 3' UTR of <i>HB</i> (Nos

Response Elements, NREs) to regulate <i>HB</i>
translation. Nos, which initially is distributed as a gradient
emanating from the posterior pole of the embryo, contains a conserved
zinc finger that mediates nonspecific RNA binding. Nos is selectively
recruited into a ternary complex on <i>HB</i>
mRNA by NRE-bound Pum. The mechanism by which the resulting Nos/Pum/NRE
complex regulates translation is not yet understood, although
deadenylation is thought to play a role (Sonoda, 2001 and references
therein). </p><p>To identify targets or cofactors of the Nos/Pum/NRE ternary
complex,  a yeast 'four-hybrid' experiment was performed; a Gal4
activation domain fusion library was screened for proteins that interact
with the ternary complex. The bait contained the RNA-binding domain of
Pum, full-length Nos, and NRE-bearing RNA. As anticipated, factors that
interact with individual components in isolation were identified.
However, one factor, which proved to be a fragment of <a href="http://www.sdbonline.org/fly/dbzhnsky/brat1.htm">Brain tumor</a>
(Brat), interacts only with the ternary complex and not with either Nos
alone, Pum alone, or a Pum/NRE binary complex. Deletion analysis
revealed that recruitment of Brat is dependent on the conserved
carboxy-terminal domain of Nos that mediates its interaction with Pum
on <i>HB</i> mRNA, and not the amino-terminal domain of Nos that mediates interaction with <a href="http://flybase.bio.indiana.edu:82/.bin/fbidq.html?FBgn0000392">Cup</a>
during early oogenesis. Mutational analysis further showed that a
fragment of Brat consisting of little more than the NHL domain is
recruited to the ternary complex. Protein-protein interaction
experiments show that Nanos and Pumilio are required to recruit Brat to <i>HB</i> mRNA and genetic experiments show that Brat is required for repression of <i>HB</i> mRNA  (Sonoda, 2001).



    

</p><p>A model is presented of how Nos, Pum, and Brat act to regulate gene

expression. The model involves combinatorial interactions among <i>cis</i>-acting
sequences in regulated mRNAs, proteins that recognize these sequences,
and the NHL domain of Brat. Recruitment of Brat occurs through
protein-protein interactions with RNA-bound Pum and Nos; formation of
the resulting quaternary complex is essential for translational control
of HB. Recruitment of Brat to the NRE jointly by Nos and Pum is
essential for regulation of <i>HB</i> mRNA. Three lines of evidence show that the NHL domain plays a key role in this process:  (1) the NHL domain is

sufficient to mediate interaction with the Nos/Pum/NRE complex, thereby

targeting Brat to <i>HB</i>
mRNA; (2) single amino acid substitutions within the NHL domain
attenuate interaction with the ternary complex and regulation of <i>HB</i> in vivo; (3) maternal expression of the wild-type NHL domain alone is sufficient to restore <i>HB</i> regulation in <i>brat</i><sup>fs</sup>
mutant embryos. This result suggests that the NHL domain contains
intrinsic translation regulatory activity. However, activity of the
isolated NHL domain is (necessarily) assayed in the presence of Brat<sup>fs</sup> mutant protein, and thus, the possibility that the amino-terminal BCC domain participates somehow in <i>HB</i> mRNA regulation cannot be ruled out (Sonoda, 2001).





</p><p>Translation regulation plays an essential role in the differentiation and development of animal cells. One

   well-studied case is the control of <i>Hunchback</i>
mRNA during early Drosophila embryogenesis by the trans-acting factors
Pumilio, Nanos, and Brain Tumor. This study reports a crystal structure
of the critical region of    Pumilio, the Puf domain, that organizes a
multivalent repression complex on the 3' untranslated region of <i>Hunchback</i> mRNA. The similarity between Pum RBD and that of another translation regulator FBF, which binds to the 3'UTR of <i>fem-3</i> mRNA in <em>C. elegans</em>,
defines a Puf (Pum and FBF) domain, which is conserved in organisms as
diverse as plants, yeast, and humans. The Puf domain is characterized
by eight imperfect repeats of ~36 amino acids (Puf repeats), followed
by a C-terminal extension. All eight repeats appear to be required for
proper folding of the Puf domain, since limited proteolysis fails to
yield stable smaller fragments. The Puf domain is thus amongst the
largest sequence-specific RNA binding motifs to be discovered; the RRM,
the KH domain (70 residues), and the dsRBD (65 residues), are much
smaller.  The PUF domain structure reveals an extended, rainbow shaped
molecule, with tandem helical repeats that bear unexpected resemblance
to the armadillo repeats in beta-catenin and the HEAT repeats in
protein phosphatase 2A. Based on the structure and genetic experiments,
putative interaction surfaces for Hunchback mRNA and the cofactors
Nanos and Brain Tumor are identified. This analysis suggests that
similar features in helical repeat proteins are used to bind extended
peptides and RNA (Edwards, 2001). </p><p>Two lines of evidence from mutagenesis studies support the
idea that the Pum concave surface binds RNA.  (1) A gene encoding the
322 residue minimal Pum RBD was randomly mutagenized and variants were
isolated that    bind normally to the wild-type NRE in yeast.
Collectively, these variants bear substitutions at 61 residues, 55 of
which map to the structure; the remaining six are in the putative 9th
repeat, not in this crystal structure. Of these, only 3 (presumably
silent) substitutions fall on the solvent exposed concave    surface,
with the remaining 52 lying elsewhere. The relative paucity of
substitutions within the inner surface is consistent with this being
the area that contacts the RNA.  (2) Based on the structure, single
substitutions were introduced in solvent-exposed residues along the
inner surface in five of the eight Puf domains and RNA binding activity
was tested in yeast. Each of these mutants is inactive. Thus, the
concentration of positive charge and the distribution of both silent
and inactivating substitutions together suggest that the RNA interacts
with the inner concave surface (Edwards, 2001). </p><p>It is proposed that <i>HB</i> mRNA binds to this inner surface
in an extended single-stranded conformation. Algorithms that predict
RNA structure suggest the NRE does not adopt a stable secondary or
tertiary structure. The minimal NRE for high affinity Pum binding
consists of nucleotides 3-27, which bracket specific contacts with
nucleotides 9, 11-13, and 21-24. The length of this minimal NRE, in an
extended single-stranded conformation (112 Å), agrees approximately
with the contour length (90 Å) of the concave surface of the Puf
domain. It is noteworthy that beta-catenin also has the highest
concentration of positive charge within its concave surface (or
groove), which is the proposed binding site for segments of cadherins,
APC, and members of the LEF-1/TCF family of transcription factors. A
recent crystal structure of a beta-catenin/TCF complex shows the TCF
segment tethered along the positively charged groove. In the case of
karyopherin-alpha, the concave surface is the binding site for the NLS
peptide. Taken together, the binding of ligands to concave surfaces is
a recurring theme in helical repeat proteins. The Pum Puf domain shows
that this type of extended surface can be used to bind RNA, as well as
peptides (Edwards, 2001). </p><p>Repression of <i>HB</i> mRNA depends not only on Pum, but also
on the recruitment of Nanos and Brat to form a quaternary complex.
Previous work suggested that Nos is recruited via residues in Puf
repeat 8. These residues map to the extra long loop between helices H1
and H2 in repeat 8, that is the main protrusion from an otherwise
relatively smooth outer Pum surface. Two different insertions into this
loop have no effect on Pum-RNA binding but eliminate recruitment of
Nos. To further define the Nos interaction surface, the collection of
Pum mutants that bind normally to RNA was tested for Nos recruitment in
yeast. Of the 61 substitutions distributed throughout the domain, only
two abrogate interaction with Nos. One is a substitution in the
putative ninth Puf repeat that is not represented in this structure,
while the other changes the solvent exposed phenylalanine on the H1/H2
loop to a serine (F1367S). Thus, the Pum surface that interacts with
Nos appears to be limited to a small region that includes the eighth
repeat and the C-terminal tail. If this tail indeed does fold into a
ninth Puf repeat, then the Pum-Nos interface would span a length of
~15-20 Å on the outer convex surface. It is tempting to think that the
C-terminal tail may only fold when Pum binds to the RNA, thereby
explaining why Nos is only recruited to the Pum/NRE binary complex and
not to Pum alone. The insertions into the long flexible loop in repeat
8 may modify its conformation such that F1367 is no longer exposed for
interaction with Nos. The proposed Phe-Nos interaction is reminiscent
of the way in which a solvent exposed phenylalanine on the receptor CD4
interacts with the HIV gp120 glycoprotein (Edwards, 2001). </p><p>Development of the Drosophila abdomen requires repression of maternal <i>Hunchback</i> (<i>HB</i>) mRNA translation in the posterior of the

embryo. This regulation involves at least four components: nanos response elements within the <i>HB</i>
3' untranslated region and the activities of Pumilio (Pum), Nanos
(Nos), and Brain tumor. To study this regulation, an RNA injection
assay was developed that faithfully recapitulates the regulation of the
endogenous <i>HB</i>
message. Previous studies have suggested that Nos and Pum can regulate
translation by directing poly(A) removal. RNAs that lack a poly(A) tail
and cannot be polyadenylated and RNAs that contain translational
activating sequences in place of the poly(A) tail are still repressed
in the posterior. These data demonstrate that the poly(A) tail is not
required for regulation and suggest that Nos and Pum can regulate <i>HB</i>
translation by two mechanisms: removal of the poly(A) tail and a
poly(A)-independent pathway that directly affects translation
(Chagnovich, 2001). </p><p>Nos, Pum, and Brat proteins regulate maternal <i>HB</i> translation in the posterior of the Drosophila embryo via the Nos response elements (NREs) in the <i>HB</i> 3' UTR. However, it has been difficult to study the mechanism of regulation in detail because <i>HB</i>
translational regulation has not been recapitulated faithfully in  in
vitro  or tissue culture assays. Recently, a number of  in vitro 
translation systems from Drosophila embryos have been established to
study the regulation of <i>nos</i> and <i>oskar</i> mRNAs. Unfortunately, it has not been possible to recapitulate <i>HB</i>
regulation with these and similar systems. RNA injection assays are a
method of choice for analyzing translational efficiency  in vivo  in a
variety of WT and mutant backgrounds by using modified RNAs. To apply
this system to Drosophila embryos and increase the sensitivity of the
injection assay, F-Luc and R-Luc genes were used as reporters. In this
injection assay, two  in vitro-transcribed RNAs are coinjected into
either the anterior or posterior pole of the preblastoderm embryo. The
control mRNA contains the <i>Renilla reniformis</i> luciferase (R-Luc)
coding region with short 5' and 3' UTRs that lack regulatory elements.
This unregulated mRNA is used to normalize for the amount of RNA
injected. The test mRNA (HFH) contains the maternal <i>HB</i> 5' UTR, the <i>F</i>-Luc coding region, and the maternal <i>HB</i>
3' UTR. After incubation, F-Luc and R-Luc activities are determined. The
relative translational efficiency of each mRNA in the anterior versus
the posterior of the embryo (A/P) then is calculated. An A/P of 1 means
the test mRNA is translated equally well in the anterior and the
posterior, whereas a ratio of greater than 1 means the test mRNA is
translated more efficiently in the anterior than the posterior.
Translation of a test mRNA with an intact <i>HB</i> 3' UTR and a WT NRE (HFH<sub>WT</sub>
A25) is repressed in the posterior versus the anterior of WT embryos
(A/P 7.8 ± 2.0), mirroring the regulation of endogenous maternal <i>HB</i> mRNA (Chagnovich, 2001).   



</p><p>To determine whether injected RNAs require the same factors for regulation as the endogenous maternal <i>HB</i> transcript, the effect of <i>nos</i>, <i>pum</i>, and NRE mutations on the translation of the HFH mRNAs was studied. HFH transcripts injected into <i>nos</i> or <i>pum</i>
mutant embryos are translated equally well in the anterior versus the
posterior. This finding is consistent with the requirement for Nos and
Pum in regulating <i>HB</i> translation  in vivo. Further, HFH
transcripts containing mutant NREs in which the six guanosines have
been changed to uracil (GU) show no significant difference in
translation between the anterior and posterior of the embryo,
regardless of the genetic background. This NRE mutation disrupts Pum
binding to the NRE  in vitro  and eliminates translational repression
of maternal <i>HB</i> mRNA  in vivo. Thus factors that are required for the regulation of the endogenous maternal <i>HB</i> transcript also are required for the regulation of the injected transcripts (Chagnovich, 2001).  



</p><p>Removal of the poly(A) tail is the prevailing model for translational regulation of many maternal mRNAs. Indeed, maternal <i>HB</i> mRNA is polyadenylated in the cytoplasm concomitant with its translational activation. In the posterior, maternal <i>HB</i>
is rapidly deadenylated in a Nos, Pum, and NRE-dependent process. To
determine whether deadenylation could account for the regulation of
maternal <i>HB</i> translation, the effect of the poly(A) tail on HFH
translation was examined in the injection assay. HFH mRNAs that lack a
poly(A) tail are polyadenylated upon injection and more efficiently
translated in the anterior versus the posterior (A/P 7.4 ± 0.8).  HFH
reporter mRNAs without a poly(A) tail were prepared that carry a point
mutation in the polyadenylation signal (AAUAAA to AAUACA) and thus
cannot be polyadenylated. These HFH mRNAs are still regulated (A/P 3.5
± 0.7), although polyadenylation is no longer detectable on transcripts
bearing this mutation. To confirm this finding,  HFH mRNAs were
injected that lack a poly(A) tail and were end-labeled with cordycepin,
an ATP analog  that lacks a 3' hydroxyl group and, as a result, blocks
further elongation of poly(A) tails. These mRNAs are also
differentially regulated in the absence of a poly(A) tail (A/P 3.0 ±
1.3). This finding suggests that a poly(A) tail is not absolutely
necessary for the regulation of <i>HB</i> translation in the embryo and that the process of deadenylation is not required for <i>HB</i>
repression. HFH mRNAs that are not polyadenlyated are less well
regulated than reporters with a poly(A) tail (A/P ~3 as compared with
A/P ~7). Because RNAs lacking a poly(A) tail are less well translated
it is possible that this difference in regulation is caused by lower
translational activation of unadenylated RNA in the anterior rather
than to less efficient repression of this RNA in the posterior
(Chagnovich, 2001). </p><p>Injected mRNAs lacking a poly(A) tail are poorly translated
and thus may not accurately reflect regulation  in vivo. Reporter mRNAs
were therefore developed that are efficiently translated independent of
a poly(A) tail. To accomplish this HFH reporter mRNAs were synthesized 
that contained the Drosophila histone <i>H1</i> 3' terminal stem loop
(HSL) in place of the poly(A) tail (HFH HSL). Most histone mRNAs do not
contain a poly(A) tail, but rather end with this conserved stem-loop
structure. It has been shown that the HSL with the stem loop-binding
protein regulates the stability and translational activity of histone
mRNAs as does the poly(A) tail  for other cellular transcripts. To
prevent addition of a poly(A) tail to the HSL, the AAUAAA
polyadenylation signal was mutated as described. HFH<sub>WT</sub> HSL
mRNAs are translated more efficiently in the anterior than the
posterior, demonstrating spatial regulation similar to the endogenous
maternal <i>HB</i> (A/P 5.4 ± 2.0). Further, this regulation depends on
an intact NRE and the presence of Nos activity. There are several
possible explanations for this finding: the NRE complex can direct
removal of the HSL; HSL-containing messages are destabilized in an
NRE-dependent manner; or the NRE complex can directly repress
translation (Chagnovich, 2001). </p><p>To determine whether the NRE represses translation of
HSL-containing mRNAs by directing removal of the HSL, just as it
directs removal of the poly(A) tail, radiolabeled, m<sup>7</sup>GpppG-capped <i>HB</i>
3' UTRs containing either a poly(A) tail or the HSL were injected into
WT embryos. Consistent with previous findings, RNAs containing a
poly(A) tail maintain the poly(A) tail in the anterior, but the poly(A)
tail is rapidly removed in the posterior. However, when RNAs containing
the HSL are injected, the HSL is maintained in the anterior and
posterior of the embryo, demonstrating that the NRE complex is not
directing removal of HSL. Next tested was whether the differences in
translation of the HSL-containing reporters were caused by differential
stability of the mRNAs. In these experiments, the levels of HSL or
poly(A)+ reporter RNAs compared with a poly(A)+ control RNA containing
a mutant NRE were examined. The relative levels of the reporter mRNAs
are not significantly altered in the anterior versus the posterior for
either the polyadenylated or the HSL-containing mRNAs. Together these
data demonstrate that translational regulation of <i>HB</i> does not
require removal of the poly(A) tail. It is concluded that NRE-directed
repression can be independent of the poly(A) tail (Chagnovich, 2001).
 </p><p>Regulation of poly(A) and HSL containing reporters is
quantitatively different, suggesting that there may be differences in
some aspects of their regulation. To further compare the translational
regulation of polyadenylated and HSL-containing mRNAs, the regulation
was followed of HFH reporter mRNAs containing a poly(A) tail versus a
HSL at 30, 60, and 90 min after injection. Whereas absolute production
of the reporters rises with time, the A/P ratio of test mRNAs
containing a poly(A) tail continues to increase up to the 90-min time
point. In contrast, the regulation of the HSL-containing transcripts
reaches a plateau by 60 min. One explanation for this disparity is that
the HSL-containing transcripts are translationally active in both the
anterior and the posterior and that active repression has to be
maintained in the posterior throughout the test period. In contrast,
poly(A)-containing mRNAs are translated at a constant rate at the
anterior, but are translationally inactivated in the posterior. To
evaluate the stability of the injected mRNAs at the later time point,
RNA was isolated at various times after injection and the levels of the
reporter mRNAs was determined. There is no significant difference in
the stability of the mRNAs containing a WT NRE (HFH) versus those
containing a mutant NRE (HCH). This suggests that poly(A) removal
inactivates the <i>HB</i> mRNA, targeting it for translational silencing but not for immediate decay tail (Chagnovich, 2001).       



</p><p>It is proposed that the NRE has two effects on <i>HB</i>
regulation: poly(A) removal and poly(A) independent translational
repression. One potential mechanism to achieve both of these effects
would be through the action of the cap-dependent poly(A) nuclease PARN.
Human and  Xenopus PARN has been shown to compete with eIF4E for
binding to the cap, thereby potentially disrupting translational
initiation. Once bound to the cap, PARN rapidly removes the poly(A)
tail. The NRE complex could recruit PARN to the RNA or destabilize the
cap complex, allowing PARN access. To test this possibility, RNAs
corresponding to the <i>HB</i> 3' UTR were synthesized that were capped with either m<sup>7</sup>GpppG
or the cap analog ApppG. mRNAs with an ApppG cap are not efficiently
deadenylated by PARN. Similarly to what is seen with m<sup>7</sup>GpppG-capped
messages, ApppG-capped messages are deadenylated in the posterior but
not the anterior. Additionally, there is no clear PARN homolog  in the
Drosophila genome. It is concluded that PARN or a PARN-related
mechanism is not mediating the effects of the NRE on <i>HB</i> mRNA (Chagnovich, 2001).  



</p><p>To determine whether the cap complex is required for regulation,
ApppG-capped HFH reporter mRNAs were injected. These mRNAs were very
poorly translated (&gt;20 times less well than the m<sup>7</sup>GpppG-capped
reporters). To obtain luciferase levels that were above background,
higher levels of reporter mRNAs (500 pg/µl) were injected. At these RNA
levels, m<sup>7</sup>GpppG-capped mRNAs were differentially regulated
(A/P 4.9 ± 1.1). Regulation of ApppG-capped reporters bearing a WT NRE
(2.2 ± 0.9) was not significantly different from that for ApppG-capped
reporters bearing a mutant NRE. These data may indicate a requirement
for the cap in NRE-mediated regulations. It is likely, that because of
the low translation levels of ApppG-capped mRNAs, the failure to
differentially regulate these mRNAs results from a failure to
efficiently activate translation in the anterior as opposed to repress
translation in the posterior (Chagnovich, 2001). </p><p>Thus, the RNA injection assay has shown that the NRE and its associated factors can repress <i>HB</i>
translation independent of the poly(A) tail and in the presence of a
heterologous translation activation signal (the HSL). At least two
models are compatible with this scenario. In the first, the NRE complex
may disrupt a single molecule or step in translation that results in
both poly(A) removal and translational repression. Because of its
central role in linking the 5' cap complex with the 3' poly(A) tail,
PABP would be a likely target in this model. In this scenario, the NRE
complex recruits factors to the <i>HB</i> RNA that could modulate PABP
activity. Drosophila Paip-2, for example, could be such a factor.
Recently, human Paip-2 has been shown to interact with PABP and disrupt
its binding to the poly(A) tail, resulting in an increase in
deadenylation and a decrease in translation. However, because a poly(A)
tail is not absolutely required for <i>HB</i> translational repression,
it is more likely that the NRE complex would disrupt the cap complex,
and that deadenylation is a consequence rather than the cause of cap
complex disruption. It has been demonstrated in plant extracts that
interaction between eIF4F and eIF4B increases poly(A) binding by
stabilizing PABP on the tail. Thus it is possible that disruption of
the cap complex or its interaction with PABP could result in decreased
translation and increased deadenylation (Chagnovich, 2001). </p><p>In an alternative model, the NRE could act by two discrete
pathways: one directly interfering with translation and a second
partially redundant pathway directing poly(A) removal. The data as well
as the absence of an obvious homolog in the Drosophila genome suggest
that the cap-dependent nuclease PARN is not the deadenylase. However,
the Drosophila genome does contain CCR4 and CAF1, which have been shown
to be components of the major cytoplasmic deadenylase in yeast, and
PAN2 and PAN3, which have a role in both nuclear and cytoplasmic
deadenylation. Either or both of these complexes could be recruited to
the <i>HB</i> mRNA by the NRE complex, thus fulfilling the
deadenylation pathway of this model. This study shows that the NRE
complex can directly repress translation of mRNAs containing the HSL.
The stem loop is a conserved structure that fulfills many of the
functions the poly(A) tail serves on other transcripts and is essential
for translation of histone mRNA. Recently, stem loop-binding protein
has been identified as a sequence-specific factor that interacts with
the stem loop and participates in histone mRNA processing and
mobilization onto polyribosomes. The mechanism of translational
activation by the stem loop and its associated factor(s) remains
unclear, although it is believed to mediate interaction between the
termini, possibly through an interaction with eIFs. It is speculated
that the NRE complex is directly interfering with translation at the
same step for mRNAs bearing a histone stem loop or poly(A) tail. Such a
common step would likely involve initiation or elongation of the
polypeptide rather than PABP. In addition, silencing of polyadenylated
RNAs would increase the efficiency of translational repression
(Chagnovich, 2001). </p><p>Although the best-studied examples of translational regulation
require removal of the poly(A) tail, examples of poly(A)-independent
regulation have been described. For example, human ribonucleoproteins K
and E1 repress translation of erythroid 15-lipoxygenase mRNA by
inhibiting assembly of the 80S ribosome. In  Xenopus, it has been shown
that Maskin interacts with eIF4E, thereby disrupting cap complex
assembly. Although Nos has two potential eIF4E binding sites, deletion
of these sites does not affect Nos regulation of <i>HB</i>. Nos,
ectopically expressed in the fly eye, can repress expression of
NRE-containing mRNAs that initiate translation via an internal ribosome
entry site, suggesting that Nos, Pum, and Brat may act downstream of
the cap. Although the possibility cannot be excluded that these
transcripts are degraded in the eye primordia as a result of
NRE-mediated removal of the poly(A) tail, it is intriguing to speculate
that the NRE complex may act to directly interfere with the function of
the general translation machinery on transcripts containing NREs.
Interestingly, other maternal RNAs (e.g., <i>Oskar</i> and <i>Nanos</i>) whose translational repression, like <i>HB</i>,
is mediated by sequences in the 3' UTR, do not require the poly(A) tail
for their regulation. This finding suggests that direct inhibition of
the translation machinery may be a common strategy in the Drosophila
embryo (Chagnovich, 2001). </p><p></p></dd></dl><hr>

<!--Copy links here--> 

 <i>hunchback</i>:<font size="-1">

<a href="http://www.sdbonline.org/fly/segment/hunchbk1.htm#Bio">Biological Overview</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk2.htm#EvHoms">Evolutionary Homologs</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk3.htm#Regul">Regulation</a> 

| <a href="http://www.sdbonline.org/fly/segment/hunchbk7.htm#TofA">Targets of activity</a> 



| <a href="http://www.sdbonline.org/fly/segment/hunchbk4.htm#PROT">Protein Interactions</a> 







| <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#DevBio">Developmental Biology</a> 

 | <a href="http://www.sdbonline.org/fly/segment/hunchbk5.htm#EfMu">Effects of Mutation</a>



 | <a href="http://www.sdbonline.org/fly/segment/hunchbk6.htm#Ref">References</a></font>



<p><a href="http://www.sdbonline.org/fly/aimain/1aahome.htm">Home page</a>: The Inter<i>active</i> Fly © 1995, 1996 Thomas B. Brody, Ph.D.</p><p> Please e-mail comments/corrections to <a href="mailto:brodyt@codon.nih.gov"><em>brodyt@codon.nih.gov</em></a>

<br>

</p><p><a href="http://www.sdbonline.org/"><img src="hunchbk8_files/SDBSmLogo.gif" alt="SDB Logo" hspace="3" vspace="3" height="50" width="50" align="middle">SDB home page</a> || <a href="http://www.sdbonline.org/google.search.html">Search the SDB site </a> 

</p></body></html>